## Introduction

Antimicrobials, substances which can kill or inhibit the growth of microbes [@doi:10/dbhq], are key effectors in the ecology of microorganisms.
The evolution of resistance to these substances, known as antimicrobial resistance (AMR), is an important and ancient process [@doi:10/b3wbvx].
In the last century we have adopted and used antimicrobials to great effect in clinical medicine and agriculture.
However, the use and abuse of antimicrobials has led to increasing levels of observed AMR [@doi:10/f9wbjs].
AMR poses a growing global health risk by undermining our ability to treat infectious diseases and perform surgeries [@doi:10/gfnrkj].
Infections by resistant microbes have greater associated mortality and morbidity [@doi:10/gfnrkj], with the European Centre for Disease Prevention and Control estimating that 25,000 people per year die due to resistant infections [@doi:10/c9zd].

To mitigate the spread of AMR, the World Health Organization (WHO) (and national government bodies e.g., [@url:canadafedaction]) have identified that strengthening surveillance of AMR, especially transmission of AMR between environmental and clinical samples [@url:whoplan], is vital.
The WHO's priority list of antibiotic resistant bacteria [@url:whopriority] highlights the multi-drug resistant pathogens upon which these efforts should be focused.
This includes the ESKAPE pathogens: _Enterococcus faecium_, _Staphylococcus aureus_, _Klebsiella pneumoniae_, _Acinetobacter baumannii_, _Pseudomonas aeruginosa_, and _Enterobacter_ spp. [@doi:10/dzzh,@doi:10/f9h4q3,@doi:10/frhgmx].
These pathogens have become resistant to many common classes of antimicrobials including penicillins, all generations of cephalosporins, gylcopeptides, quinolones, and peptide antibiotics [@doi:10/dzzh].
Concerningly, this also includes resistance to last-line carbapenem and colistin antibiotics related to mobilised AMR genes i.e., laterally transferable between unrelated lineages [@doi:10/d595m9].

Carbapenems are broad spectrum beta-lactam antibiotics that cannot be degraded by most beta-lactamases, as such they are used to treat life threatening, high risk, and multi-drug resistant bacterial infections [@doi:10/c9zd].
The number of observed carbapenem resistant _K. pneumoniae_, _E. coli_, _E. cloacae_, and _A. baumanii_ hospital-acquired infections have increased in Canada since 2013 [@url:canadanosocomial], presenting a new threat to public-health.
Resistance to carbapenems is largely mediated by acquisition of one of several families of carbapenemase genes [@doi:10/c9zd].
Due to their mobility and host-range, 3 of the carbapenemase gene families of most pressing public-health concern are _Klebsiella pneumoniae_ carbapenamase (KPC), New Delhi metallo-beta-lactamase (NDM), and OXA-48-like beta-lactamases [@doi:10/c9zd].
These are, respectively, the most prevalent carbapenemase families within the A, B, and D beta-lactamase classes [@doi:10/c9zd].

For carbapenem resistant infections, polymyxin E (colistin) and B antibiotics are frequently the only remaining option.
Polymyxins act by disrupting membrane permeability [@doi:10/dbht] and have had limited use in humans due to their neurotoxic and nephrotoxic effects [@doi:10/dbht].
Despite this, mobilized colistin resistance (MCR) has emerged in clinical pathogens [@doi:10/gdqcfq] likely due to transmission from agricultural usage of colistin [@pmid:28264015].
This underscores the vital importance of a "One-Health" based comprehensive surveillance program of AMR genes and their movement across environmental, agricultural, and clinical contexts.

This kind of AMR surveillance requires large-scale phylogenetic analyses which characterise the total diversity, context, and spread of these key resistance determinants.
In order to perform these types of analyses we need two resources: a clear consolidated definition of what constitutes an AMR-related gene and a large number of sequenced microbial genomes.
However, naming and defining AMR genes is difficult due to a complex legacy of historical nomenclature (prior to widespread DNA sequencing), and a huge quantity of active AMR research from a broad-range of stakeholders including academia, government organizations, human health, animal health, and agri-food industries [@url:canadafedaction].

AMR determinants are typically classified into families [@doi:10.1093/nar/gkz935], some of which are based on phenotypic properties, and others on sequence homology.
These families are usually named according to a specific unifying characteristic, such as the resistance conferred (e.g., OXA-48-like, MCR), the geographic (e.g., "New Delhi" portion of NDM) or taxonomic (KPC) source of first observed examples, or the molecular structure ("metalloenzyme" portion of NDM).
These are reduced to a short acronym or abbreviated version (i.e., a gene symbol) and assigned a numerical suffix to distinguish variants.
Each allele, determined to be novel by some (ideally evolutionary) criterion, is assigned a new suffix number in this scheme.
For example, the MCR phosphoethanolamine transferase gene family inhibits the binding of colistin to the cell membrane.
The MCR-3 gene has a sequence identity which has been deemed to be significantly different from that of the other MCR genes in the family.
MCR-3.1, in turn has been deemed to be a variant of the MCR-3 gene, with only slight variation from other MCR-3 genes [@doi:10/dw7v].
Alternatively, members of the NDM beta-lactamase gene family are named using a different set of criteria.
Specifically, genes that differ by only a single amino acid substitution are assigned a distinct suffix e.g., NDM-1 and NDM-2.
Consequently, the amount of diversity encompassed within different gene names and alleles is hugely variable across AMR families.
This problem is further exacerbated by the existence of AMR families such as the OXA-beta lactamases which are not defined on sequence identity but phenotypically.
Within this "family", there are numerous homologous sets of genes unified only by an ability to hydrolyse oxacillin.
OXA-20 has less than 20% amino acid sequence identity when compared with to other OXA-48-like family beta-lactamases [@doi:10/dwsr].
This highly variable complex system of classification has the potential to be misleading when conducting AMR research across the breadth of known AMR determinants.

High-quality manually curated AMR databases such as the Comprehensive Antibiotic Resistance Database (CARD) [@doi:10.1093/nar/gkz935] provide a unified resource for the definition, nomenclature and classification of AMR genes.
CARD organizes this information through the Antibiotic Resistance Ontology (ARO), a controlled vocabulary with defined relationships.
There are two datasets within CARD, CARD:Canonical and CARD:Resistomes & Variants (R&V).
Canonical is a conservative set of AMR genes and mutations that have been experimentally verified as being associated with resistance in a peer-reviewed publication.
As a consequence, most of the entries in CARD:Canonical are derived from the most intensively studied organisms, such as the ESKAPE pathogens.
On the other hand, CARD:R&V contains AMR sequences that have been detected by searching a set of 88 WHO priority organism genomes from central repositories of publicly available sequencing data such as National Center for Biotechnology Information (NCBI) for genes similar to corresponding Canonical sequences.
This _in silico_ search is performed using CARD's BLAST/DIAMOND-based Resistance Gene Identifier (RGI) tool and greatly increases the sequence diversity in CARD [@doi:10.1093/nar/gkz935].

However, this survey is still limited to a subset of all genomes currently available in central International Nucleotide Sequence Database Collaboration (INSDC) repositories like NCBI.
If we want to thoroughly understand the evolution and spread of AMR genes we need to as much genomic diversity as possible.
Unfortunately, the genomes in databases are largely sequenced from microbes that can be easily cultured. 
As only a subset of microbial diversity can be cultured, this means many of our existing genomes aren't necessarily representative of the environment from which they were sampled [@doi:10.1038/s41564-017-0083-5].
Recently, techniques have been developed that allow the recovery of genomes from metagenomic data e.g., [@doi:10.7717/peerj.1165,@doi:10.1038/s41564-018-0171-1].
As metagenomic sequencing, the direct sequencing of all DNA in a sample, doesn't require this bias inducing culturing step these metagenome-assembled genomes (MAGs) represent a huge source of previously unseen microbial diversity.
Parks et. al. [@doi:10.1038/s41564-017-0083-5], generated new 7,903 bacterial and archaeal (Uncultured Bacteria and Archaea; UBA) MAGs representing >30% increase in the sampled phylogenetic diversity of the bacteria and archaea.

By combining this MAG dataset with CARD:R&V, and other NCBI non-redundant data, we are able to greatly increase the breadth of data available for surveillance beyond clinical samples.
Phylogenetic analyses of this data was then be used to characterise the evolution and novel diversity of these critical public-health priority carbapenemase (KPC, NDM, and OXA-48) and colistin (MCR) AMR gene families.
These analyses also highlight inconsistencies and possible refinements in the classification and nomenclature of these genes and their variant alleles.
Therefore, in this work we present an in-depth investigation of the classification and phylogeny of KPC, NDM, OXA-48, and MCR gene families across all currently available sequenced genomes and MAGs.
