## Results

### Phylogenetic analysis of MCR sequences

A phylogeny with only CARD:Canonical and CARD:R&V MCR gene family sequences was created, showing the relationships without diversity from NCBI or the MAGs, including an out-group, 32 CARD:Canonical sequences, and 54 CARD:R&V sequences.
The resulting tree in Figure {@fig:canon-prev-tree} shows MCR family members 1 through 9 forming distinct sub-clades in the tree.
MCR-1, MCR-2, and MCR-6 form a closely related clade.
This clade is closely related to ICR-Mc [@doi:10/gdnrmn], another phosphoethanolamine transferase which confers colistin resistance.
MCR-3 and MCR-7, also form a closely related clade.

![Phylogenetic relationship of 32 canonical (labels prefixed with 'lcl_canon' in yellow), 54 CARD:R&V (labels prefixed with 'lcl_prev' in tan) MCR family sequences, and an outgroup from Betaproteobacteria ('lcl_outgroup' in grey). Each MCR variant is coloured based on its allele number.](images/mcr_canon_prev_full.mintax.svg){#fig:canon-prev-tree}

The tree visualisation was further condensed to only a single representative for each MCR allele (figure {@fig:canon-prev-tree-collapsed}).
Regions of the tree vary in their potential to accept additional diversity.
Some clades are tightly related, while others have more distance between them.
Newly discovered genes could some day be found to be sister to these more distantly related clades.
Regions containing more evolutionary distance are accommodating to additional diversity, while closely related regions are not.

![Phylogenetic relationship of 9 representative MCR family sequences, and an outgroup from Betaproteobacteria. (labels prefixed with 'lcl_canon' in yellow are canonical, labels prefixed with 'lcl_prev' in tan are CARD:R&V, and an outgroup from Betaproteobacteria is prefixed with 'lcl_outgroup' in grey)](images/mcr_canon_prev_collapsed.mintax.svg){#fig:canon-prev-tree-collapsed}

The phylogenies represented by Figures {@fig:mcr-3-9}, {@fig:mcr-5-icr}, and {@fig:canon-prev-nrdb-uba-tree} were created with additional diversity by querying the NCBI non-redundant database, and the MAGs from the Parks _et. al._ UBA data [@doi:10/cczd].
The filtering criteria in Table 1 were applied to these data sets.

After filtering, 409 NCBI sequence hits, labeled as phosphoethanolamine lipid A transferase, and 104 NCBI sequence hits, labeled as MCR family, were selected for addition to the phylogeny.
The 7903 draft quality MAGs were queried for AMR genes with RGI.
RGI produced 1457246 loose hits, 7171 strict hits, and 310 perfect hits.
After applying the filters from Table 1, only 63 MAG derived genes were retained for the phylogeny.
Due to the large size, the phylogeny displayed in Figure {@fig:canon-prev-nrdb-uba-tree} was subdivided into Figures {@fig:mcr-3-9}, and {@fig:mcr-5-icr}.
The subdivisions were chosen based on a pruning at the point of the most recent common ancestor of sets of canonical MCR labeled leaves.
The set including MCR-3, MCR-7, and MCR-9, and the set including MCR-1, MCR-2, MCR-5, MCR-6, and ICR-Mc, contained several clades sister to some of the canonical and CARD:R&V leaves.
Other choices for leaf sets contained added diversity that was distant from canonical sequences, and were not chosen as a focused figures.

Sister to the canonical MCR-3 sequences, and the most recent common ancestor of MCR-3 and MCR-7 clades (figure {@fig:mcr-3-9}), there is a clade of sequences from NCBI which have been reported as MCR-3.
Present within this clade is a single result from the Parks _et. al._ UBA dataset [@doi:10/cczd], UBA705.
This leaf is labelled as *Aeromonas hydrophila*, and is associated with rice paddy field-soil microbial fuel cells.
This is present within the clade alone with several other canonical and non-redundant *Aeromonas*.
*A. hydrophila* is a species which has been found to have an MCR-3 gene.

Sister to the clade containing MCR 7, and the most recent common ancestor of MCR 3 and MCR 7 clades, is a clade of NCBI sourced phosphoethanolamine lipid A transferase.
This clade consists of the genus of Gram-negative bacteria, *Aeromonas*, [@doi:10/cbnfww] which is sometimes involved in human infection.
This clade also includes an *A. veronii* hit from epidermal mucus of *Anguilla anguilla* in the UBA data.
Sister to the MCR 9 clade and the most recent common ancestor of MCR 3 and 9, a clade of *Aeromonas* associated phosphoethanolamine lipid A transferases appear.
The MCR 9 containing clade contains 8 UBAs and several NCBI non-redundant hits not reported as MCR family genes.
5 Loose hits for MCR in a *Leclercia adecarboxylata* branch within this clade.
These *Leclercia* branch as a sister to the common ancestor of MCR-9 and MCR-3 which are well supported.
The *Leclercia* UBAs were all sampled from New York City MTA subway.
*Acinetobacter* is another opportunistic pathogen [@doi:10/b2zr9k] which is becoming resistant to many antimicrobials.

![Clade containing putative MCR3 and MCR 9 clades pruned from phylogenetic relationship of 32 canonical (labels prefixed with 'lcl_canon'_ in yellow), and 54 CARD:R&V (labels prefixed with 'lcl_prev' in tan) MCR family sequences, 595 NCBI non-redundant sequences (labels prefixed with 'lcl_nrdb' in blue), and 91 UBA sourced sequences (labels prefixed with 'lcl_uba' in green). Each MCR variant is coloured based on its allele number. If the sequence is not reported to be MCR family it is coloured in grey.](images/mcr3_mcr9_diversity.mintax.svg){#fig:mcr-3-9}

Sister to the MCR 5 and ICR-Mc clades (figure {#fig:mcr-5mayb-icr}), there appear clades of diversity in the genus *Psychrobacter*.
A descendant to this ancestor, is a clade of *Psycrobacter* species bacteria.

This clade includes several hits from the non-redundant database, and two hits from the UBA data.
According to the UBA [@doi:10/cczd] data, the identity of these samples are,
UBA3068, A *Psychrobacter sp.*, sampled from Oil polluted marine microbial communities and, UBA4193, a *Psychrobacter sp.*, sampled from the New York City subway.
The quality information for these sequences, shown in table ?? [TODO: do we need a table with quality information], shows that UBA3068 is near complete, while UBA4193 is only partial.
It is encouraging to see that even with the quality difference, these two sequences branch in the expected clade.
*Psychrobacter* [@doi:10/dbpr] is a widespread genus and includes many cold adapted bacteria, it is an opportunistic pathogen, and has been found to sometimes be a cause of infections in humans, animals, and fish. Many new species of *Psychrobacter* have been discovered in cold climates [@doi:10/fd9rb9].
Some of the species have been shown to be resistant to colistin, like *P. vallis ps. nov.* and *P. aquaticus ps. nov.* [@doi:10/cdq9w4] and is sister to [TODO: display new tree such that *Acinetobacter* is shown]
Another clade sister to hese two variants contains phosphoethanolamine transferases from the genus *Stenotrophomonas*, including *S. maltophilia* and *S. acidaminiphila* which is a multidrug-resistant opportunistic pathogen [@doi:10/fxx2tt].
Several UBA hits for *S. maltophilia* show up as shotgun sequencing of environmental samples. [TODO: quality information]

![Clade containing putative MCR 5 and ICR-Mc clades pruned from phylogenetic relationship of 32 canonical labels (prefixed with 'lcl_canon'_ in yellow), and 54 CARD:R&V(labels prefixed with 'lcl_prev' in tan) MCR family sequences, 595 NCBI non-redundant sequences (labels prefixed with 'lcl_nrdb' in blue), and 91 UBA sourced sequences (labels prefixed with 'lcl_uba' in green). Each MCR variant is coloured based on its allele number. If the sequence is not reported to be MCR family it is coloured in grey.](images/mcr5_icrmc_diversity.mintax.svg){#fig:mcr-5-icr}

### Phylogenetic analysis of KPC

Using the 18 canonical sequences from Table {@tbl:kpc-variants} as query sequences, 25 CARD:R&V sequences, 376 non-redundant sequences, and 6 UBA sequences were recovered from the various databases. The result was 
combined in the phylogeny displayed in figure {@fig:kpc-tree}.
This phylogeny shows that many of the resulting sequences are distant from the canonical sequences. The main clade in figure {@fig:kpc-main-tree} shows that many of the AMR determinants are named as distinct sequences, while not having much phylogenetic distance between them.
The most closely related clade is a clade containing IMI and SME-type beta-lactamases.
The remaining sequences are in a clade containing a large amount of class A beta-lactamases from the non-redundant data. All of the UBA sequences were present in this portion of the tree.

![A clade pruned from the phylogeny in figure {#fig:kpc-tree} which represents the most closely related genes to the KPC family genes. 18 canonical labels (prefixed with 'lcl_canon'_ in yellow), and 25 CARD:R&V (labels prefixed with 'lcl_prev' in tan), and 28 NCBI non-redundant sequences (labels prefixed with 'lcl_nrdb' in blue). Each KPC variant is coloured based on its allele number. If the sequence is not reported to be KPC family it is coloured in grey.](images/kpc_main_clade.mintax.svg){#fig:kpc-main-tree}


### Phylogenetic analysis of NDM

14 canonical sequences were used to query the data for NDM beta-lactamase.
The genes retained for the phylogeny were 14 canonical sequences, 8 CARD:R&V sequences, 12 non-redundant sequences, 1 UBA sequence, and an out-group.
This resulted in the phylogenetic relationship in Figure {@fig:ndm-tree}.

![Phylogenetic relationship of The NDM genes with 14 canonical (labels prefixed with 'lcl_canon' in yellow), 8 CARD:R&V (labels prefixed with 'lcl_prev' in tan), 12 NCBI non-redundant database sequences (labeled as 'lcl_nrdb' in blue), 1 UBA sequence (labeled as 'lcl_uba' in green), and an outgroup from Betaproteobacteria (labeled as 'lcl_outgroup' in grey). Each NDM variant is coloured based on its allele number.](images/ndm_full.mintax.svg){#fig:ndm-tree-all}

The 12 sequences resulting from the NCBI data included 3 genes reported to be NDM-1, NDM-3, and NDM-5, while the other 4 genes were reported to be general beta-lactamase/carbapenemase hits.
As in the KPC phylogeny (figure {@fig:kpc-main-tree}), there is very little phylogenetic difference between the named variants of NDM.
The only phylogenetically distinct genes in this clade were the CARD:R&V hit for NDM-1, and the NCBI hits labeled as NDM-1 and NDM-5.
The naming of these sequences seem not to be congruent with the phylogenetic relationships present in the tree.
Some of the NDM-1 and NDM-5 genes are more closely related to the other NDM variants than themselves.
One BLAST result from the UBA data branches far from the clade containing the canonical sequences.
This indicates that under the coverage queried, there are no reasonably detectable NDM homologues in the UBA data.
The alignment of this UBA under this relaxed query coverage of 60% (see Table 1) is questionable.
Reducing query coverage further could produce more sequences from the UBA data, but the resulting alignment would produce a meaningless phylogeny.

### Phylogenetic analysis of OXA-48

In querying for the OXA-48 portion of the analysis, multiple BLAST results for UBA sequences were found, but the hits were too phylogenetically dissimilar to draw a conjecture about their relationship to the OXA family. 6 canonical sequences, and the 14 CARD:R&V sequences, and 20 non-redundant sequences are combined in the phylogeny in Figure {@fig:oxa-tree}

![The main phylogenetic relationship pruned from the tree inferred from OXA-48-like sequences from figure {#fig:oxa-tree}. 6 canonical (labels prefixed with 'lcl_canon'_ in yellow), and 14 CARD:R&V (labels prefixed with 'lcl_prev' in tan), 20 NCBI non-redundant sequences (labels prefixed with 'lcl_nrdb' in blue). Each OXA-48 variant is coloured based on its allele number. If the sequence is not reported to be OXA-48 family it is coloured in grey.](images/oxa_main_clade.mintax.svg){#fig:oxa-main-tree}

The OXA-48 CARD:R&V hits added further diversity to the reference OXA-48-like sequences. OXA-436 [@doi:10/db7x] was found, clustered with no other gene, in the CARD:R&V data, and OXA-514 and OXA-515 were found in the non-redundant data. The named variants are too similar to produce significant phylogenetic resolution.
