## Discussion

### An expanded set of candidate resistance genes

Homologous sequences from NCBI, CARD:R&V data, and a large set of MAGs, were queried, filtered, and added to phylogenies.
The filtering criteria were chosen to ignore small changes between sequences (99-100% sequence identity threshold), and to produce meaningful alignments in the final phylogenies (90-99% query coverage).

NCBI and CARD:R&V added new homologues to the phylogenies, however, these conservative filtering criteria contributed obvious novel homologues from only the MCR MAG analysis, and not in the three beta lactamase MAG analyses.
This difference could be explained by plasmid distributions.
Plasmids could have been poorly recovered in the assembly phase of the SRA experiments, many of which produce short reads (Illumina HiSeq 2000 and 2500).
TODO: Look at The number/percentages of completely sequenced genomes, completely sequenced plasmids analyzed for each pathogen, and see if there is any concrete evidence that the above is actually happening.
This seems to be problematic with bias in sampling, so other ideas/sources that may be promising are welcome.
Plasmids from short reads are known to be difficult to assemble[@doi:10/gf6b63] [@doi:10/gf6b63] [@doi:10/gdf329].

### Phylogenetic positioning of AMR genes from NCBI and MAGs

Another way in which the analyses of MCR and the beta lactamases differ is in the spread and diversity of the relationships.
In addition to MAG sources, which are completely absent from the beta lactamase trees, the MCR tree contains several sequences from NCBIâ€™s non-redundant database which land close to canonical clades.
Some of the added NCBI and MAG sourced sequences are not so close, but share ancestors with canonical sequences nonetheless.
This suggests that there could be more diversity to be found within the MCR class.
In contrast, the added diversity in phylogenies from OXA-48, KPC is distant from the canonical and CARD:R&V sequences.
The matches which are close, are nearly identical to the canonical and CARD:R&V sequences.
The beta-lactamase phylogenies are low in gene diversity, and are more tightly defined than the MCR phylogenies.

### Phylogenetic relationships vs. nomenclature

In assessing the value of the relationships present in these more tightly related phylogenies, it is surprising that the literature has produced so many named variants.
In the tree for NDM, we have named variants numbered from NDM 1 to NDM 14, yet most of the tree would be more appropriately displayed with multifurcating nodes.
There are fifteen named variants, but there are only three weakly distinct clades, and little bootstrap support (TODO: look at SNPs put this in results?).
The most divergent NDM sequence is a CARD:R&V sequence which is labelled as NDM-1, yet is more distant from other homologs named NDM-1 than other NDM-1 genes.
The criteria for naming these genes are seemingly arbitrary.
It is also possible that incentive to find new names for producing publication contributes to many of the more redundant variants.
Similarly, KPC is very tightly related and some of the same arguments could be made for classification of these sequences.
In the tree for OXA-48 like genes, we see more diversity, with branches of more support.
The tree forms few tightly related clades, and allows some diversity from NCBI to be added to the tree.
The OXA group, as a whole, is very diverse, but only because they are classified phenotypically.
If the OXA were more thoroughly separated into actual homologous groups, perhaps we would see more diversity added to the group, and perhaps even from metagenomic data.
These groups of OXA could behave very differently and some could have data recovery similar to that of MCR, but we are missing out on this because of the phenotypic naming scheme over a rigorous scheme which takes into account the genetic similarity. In MCR, the naming situation is less arbitrary. The closely related variants are classified under the same primary number.
Several versions of MCR-1 exist, for example, and all form a clade.

### Should naming reflect phylogenetic relationships?

Finding an ideal nomenclature scheme which within, or generalizing across AMR families is problematic.
Nomenclature which has been shown to be phylogenetically consistent, such as that of MCR, is nonetheless subject to ambiguity.
Researchers have suggested, for example for MCR [@doi:10/dw7v], using amino acid sequence identity differences rather than possibly trivial nucleotide differences, but this still subject to ambiguity in cutoff of percent identity.
Fixed cutoffs of 2% have been suggested by some researchers[@doi:10/f8gn4r], but depending on the mechanism of resistance, the level of diversity required to induce differences in substrate specificity will change between AMR genes.
One could argue that the genomic and evolutionary context should be also considered in the nomenclature [@doi:10/dw7r].
It has been suggested, for example that mobilized, and chromosomal genes differ in their evolutionary variance [@doi:10/dw7r].
In addition, the current study demonstrates that the diversity and spread varies between gene families.
This could be explained by the way MCR works by modifying the target of polymyxin, unlike the beta-lactamases, which work by destroying the beta-lactams.
Justifying naming of a new NDM gene, when all genes available to the public are essentially identical, seems to be problematic, however, further investigations into certain patterns of changes in the amino acid sequences may further granularize our classification.
Ultimately, many interesting questions about the nomenclature of these genes are rooted in ontology.
Classifying these genes is of utmost importance for communication between scientists, and a naming scheme which seeks a balance between practicality, evolutionary relationships, function, and context, must be found.
