<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-CA" xml:lang="en-CA">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Jackie Purdue" />
  <meta name="author" content="Jocelyn McDonald" />
  <meta name="author" content="Dayna Mikkelsen" />
  <meta name="author" content="Finlay Maguire" />
  <meta name="author" content="Robert G. Beiko" />
  <meta name="dcterms.date" content="2020-09-30" />
  <meta name="keywords" content="metagenome, antimicrobial resistance" />
  <title>Incorporation of metagenome-assembled genome data into antimicrobial resistance surveillance identifies inconsistencies in carbapenemase and colistin resistance gene nomenclature</title>
  <style>
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Incorporation of metagenome-assembled genome data into antimicrobial resistance surveillance identifies inconsistencies in carbapenemase and colistin resistance gene nomenclature</h1>
</header>
<p><small><em> This manuscript (<a href="https://jackiepurdue.github.io/phylogenetic-amr-survey-manuscript/v/496508e1fdcbac123907e7a200032afee5efc07b/">permalink</a>) was automatically generated from <a href="https://github.com/jackiepurdue/phylogenetic-amr-survey-manuscript/tree/496508e1fdcbac123907e7a200032afee5efc07b">jackiepurdue/phylogenetic-amr-survey-manuscript@496508e</a> on September 30, 2020. </em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Jackie Purdue</strong><br> <img src="images/orcid.svg" alt="ORCID icon" class="inline_icon" /> <a href="https://orcid.org/0000-0003-3131-0688">0000-0003-3131-0688</a> · <img src="images/github.svg" alt="GitHub icon" class="inline_icon" /> <a href="https://github.com/jackiepurdue">jackiepurdue</a> · <img src="images/twitter.svg" alt="Twitter icon" class="inline_icon" /> <a href="https://twitter.com/jackiepurdue">jackiepurdue</a><br> <small> Faculty of Computer Science, Dalhousie University </small></p></li>
<li><p><strong>Jocelyn McDonald</strong><br><br> <small> Faculty of Computer Science, Dalhousie University </small></p></li>
<li><p><strong>Dayna Mikkelsen</strong><br><br> <small> Faculty of Computer Science, Dalhousie University </small></p></li>
<li><p><strong>Finlay Maguire</strong><br> <img src="images/orcid.svg" alt="ORCID icon" class="inline_icon" /> <a href="https://orcid.org/0000-0002-1203-9514">0000-0002-1203-9514</a> · <img src="images/github.svg" alt="GitHub icon" class="inline_icon" /> <a href="https://github.com/fmaguire">fmaguire</a> · <img src="images/twitter.svg" alt="Twitter icon" class="inline_icon" /> <a href="https://twitter.com/finlaym">finlaym</a><br> <small> Faculty of Computer Science, Dalhousie University · Funded by Donald Hill Family Fellowship in Computer Science </small></p></li>
<li><p><strong>Robert G. Beiko</strong><br><br> <small> Faculty of Computer Science, Dalhousie University </small></p></li>
</ul>
<h2 id="abstract" class="page_break_before">Abstract</h2>
<p>The increase and spread of antimicrobial resistance (AMR) poses a global threat to human and animal health. The emergence of mobilised resistance to last-line antibiotics such as carbapenems and polymyxins are a particular cause for alarm. Given their ability to be laterally transferred between unrelated bacteria, surveillance of the evolution and transmission of these genes is vital to mitigating their impact. However, the poor characterisation of many important non-clinical environments and bacteria as well as failings in the existing classification and nomenclature of AMR genes pose a major barrier to this work. Metagenome assembled genome (MAG) methods, in which genomes are recovered from untargeted metagenomic sequencing, have been used to greatly increase the diversity of sampled bacterial genomes. Therefore, by profiling AMR genes detectable in MAG datasets we can greatly expand surveillance of these undersampled environments and organisms. Through this, we can also identify inconsistencies and potential refinements of the existing nomenclature.</p>
<p>In this study, the phylogenetic neighborhoods of several mobilised AMR gene families, of public-health importance, were examined for diversity, spread, and classification consistency. These gene families include: mobile colistin resistance (MCR), New Delhi beta-lactamase (NDM), Klebsiella pneumoniae carbapenemase (KPC), and the OXA-48-like beta-lactamase lactamase family. Canonical genes in the Comprehensive Antibiotic Resistance Database (CARD) for these families were used to identify putative homologues within CARD:Resistomes &amp; Variants data, NCBI sequence databases, and draft quality MAGs from a large set of uncultivated bacterial and archeal sources (UBA). For each of these 4 AMR gene families, phylogenetic analyses were conducted for the expanded set of candidate resistance genes.</p>
<p>Although limited uncharacterised diversity in NDM, KPC, and OXA-48 carbapenemases was uncovered, several possibly novel clades of MCR genes were discovered. In general, the carbapenemas families were found to be more tightly defined, with most added diversity being either extremely close, or phylogenetically distant. The MCR family showed many new clades of diversity, even within well defined “canonical” clades. A large number of existing gene names within all 4 families were identified to be inconsistent with phylogeny or inconsequential in relation to the diversity they encompass. Additionally, there was a huge variability in the diversity encompassed by individual allele names across the AMR gene families analysed. These results highlight the need to incorporate MAG data into AMR surveillance work and to develop consistent, robust, phylogenetically defined nomenclature for AMR genes.</p>
<h2 id="introduction">Introduction</h2>
<p>Antimicrobials, substances which can kill or inhibit the growth of microbes <span class="citation" data-cites="Zvf1VhKR">[<a href="#ref-Zvf1VhKR" role="doc-biblioref">1</a>]</span>, are key effectors in the ecology of microorganisms. The evolution of resistance to these substances, known as antimicrobial resistance (AMR), is an important and ancient process <span class="citation" data-cites="2xaXclNM">[<a href="#ref-2xaXclNM" role="doc-biblioref">2</a>]</span>. In the last century we have adopted and used antimicrobials to great effect in clinical medicine and agriculture. However, the use and abuse of antimicrobials has led to increasing levels of observed AMR <span class="citation" data-cites="1ByMfX8Y1">[<a href="#ref-1ByMfX8Y1" role="doc-biblioref">3</a>]</span>. AMR poses a growing global health risk by undermining our ability to treat infectious diseases and perform surgeries <span class="citation" data-cites="kDtOJsGI">[<a href="#ref-kDtOJsGI" role="doc-biblioref">4</a>]</span>. Infections by resistant microbes have greater associated mortality and morbidity <span class="citation" data-cites="kDtOJsGI">[<a href="#ref-kDtOJsGI" role="doc-biblioref">4</a>]</span>, with the European Centre for Disease Prevention and Control estimating that 25,000 people per year die due to resistant infections <span class="citation" data-cites="OLzrpsK4">[<a href="#ref-OLzrpsK4" role="doc-biblioref">5</a>]</span>.</p>
<p>To mitigate the spread of AMR, the World Health Organization (WHO) (and national government bodies e.g., <span class="citation" data-cites="10jnMhzRm">[<a href="#ref-10jnMhzRm" role="doc-biblioref">6</a>]</span>) have identified that strengthening surveillance of AMR, especially transmission of AMR between environmental and clinical samples <span class="citation" data-cites="3pJyRnMt">[<a href="#ref-3pJyRnMt" role="doc-biblioref">7</a>]</span>, is vital. The WHO’s priority list of antibiotic resistant bacteria <span class="citation" data-cites="Pb5xYOu4">[<a href="#ref-Pb5xYOu4" role="doc-biblioref">8</a>]</span> highlights the multi-drug resistant pathogens upon which these efforts should be focused. This includes the ESKAPE pathogens: <em>Enterococcus faecium</em>, <em>Staphylococcus aureus</em>, <em>Klebsiella pneumoniae</em>, <em>Acinetobacter baumannii</em>, <em>Pseudomonas aeruginosa</em>, and <em>Enterobacter</em> spp. <span class="citation" data-cites="bODq6dmJ">[<a href="#ref-bODq6dmJ" role="doc-biblioref">9</a>]</span>. These pathogens have become resistant to many common classes of antimicrobials including penicillins, all generations of cephalosporins, gylcopeptides, quinolones, and peptide antibiotics <span class="citation" data-cites="bODq6dmJ">[<a href="#ref-bODq6dmJ" role="doc-biblioref">9</a>]</span>. Concerningly, this also includes mobilised, i.e. laterally transferable between unrelated lineages, resistance to last-line carbapenem and colistin antibiotics <span class="citation" data-cites="oR3fXrzb">[<a href="#ref-oR3fXrzb" role="doc-biblioref">12</a>]</span>.</p>
<p>Carbapenems are broad spectrum beta-lactam antibiotics that cannot be degraded by most beta-lactamases, as such they are used to treat life threatening, high risk, and multi-drug resistant bacterial infections <span class="citation" data-cites="OLzrpsK4">[<a href="#ref-OLzrpsK4" role="doc-biblioref">5</a>]</span>. The number of observed carbapenem resistant <em>K. pneumoniae</em>, <em>E. coli</em>, <em>E. cloacae</em>, and <em>A. baumanii</em> hospital-acquired infections have increased in Canada since 2013 <span class="citation" data-cites="hbKzfUrw">[<a href="#ref-hbKzfUrw" role="doc-biblioref">13</a>]</span>, presenting a new threat to public-health. Resistance to carbapenems is largely mediated by acquisition of one of several families of carbapenemase genes <span class="citation" data-cites="OLzrpsK4">[<a href="#ref-OLzrpsK4" role="doc-biblioref">5</a>]</span>. Due to their mobility and host-range, 3 carbapenemase gene families of public-health concern are <em>Klebsiella pneumoniae</em> carbapenamase (KPC), New Delhi metallo-beta-lactamase (NDM), and OXA-48 type beta-lactamases <span class="citation" data-cites="OLzrpsK4">[<a href="#ref-OLzrpsK4" role="doc-biblioref">5</a>]</span>.</p>
<p>Carbapenamases are assigned to one of four classes (A-D) based on their molecular structure <span class="citation" data-cites="16ZzmUN7q">[<a href="#ref-16ZzmUN7q" role="doc-biblioref">14</a>]</span>. Class A, C, and D use a serine residue to hydrolyze beta-lactam antibiotics <span class="citation" data-cites="2p1n2mr5">[<a href="#ref-2p1n2mr5" role="doc-biblioref">15</a>]</span>. KPC genes are a part of class A, and are the most prevalent in that class <span class="citation" data-cites="OLzrpsK4">[<a href="#ref-OLzrpsK4" role="doc-biblioref">5</a>]</span>. Many variants of KPC are spreading and being discovered, however carbapenamases within this class are more rare than those within classes B, C, and D <span class="citation" data-cites="OLzrpsK4">[<a href="#ref-OLzrpsK4" role="doc-biblioref">5</a>]</span>. Class B beta-lactamases are metallo-enzymes that use a zinc active site to hydrolyze beta-lactam antibiotics. The NDM gene is classified as a class B beta-lactamase <span class="citation" data-cites="OLzrpsK4">[<a href="#ref-OLzrpsK4" role="doc-biblioref">5</a>]</span>, and is the most prevalent in that class <span class="citation" data-cites="OLzrpsK4">[<a href="#ref-OLzrpsK4" role="doc-biblioref">5</a>]</span>. Class D contains the OXA-48 beta-lactamases which are the most prevalent in that class <span class="citation" data-cites="OLzrpsK4">[<a href="#ref-OLzrpsK4" role="doc-biblioref">5</a>]</span>. Unlike KPC and NDM, the OXA family is characterized by phenotype rather than genotype. This “family” is characterized by specific hydrolases that can hydrolyze oxacillin and cloxacillin, with the enzyme being poorly inhibited by clavulanic acid (https://card.mcmaster.ca/ontology/36026). This results in relatively few homology relationships within the family. A subset of this family, OXA-48 possesses carbapenem hydrolyzing activity <span class="citation" data-cites="UV1QqTS8">[<a href="#ref-UV1QqTS8" role="doc-biblioref">16</a>]</span>. The OXA-48 subfamily consists of five variants: OXA-48, OXA-162, OXA-163, OXA-181, OXA-204, and OXA-232 <span class="citation" data-cites="FqGNAObZ">[<a href="#ref-FqGNAObZ" role="doc-biblioref">17</a>]</span>.</p>
<p>For carbapenem resistant infections, polymyxin E (colistin) and B antibiotics are frequently the only remaining option. Polymyxins act by disrupting membrane permeability <span class="citation" data-cites="17k3PVuIy">[<a href="#ref-17k3PVuIy" role="doc-biblioref">18</a>]</span> and has had limited use in humans due to its neurotoxic and nephrotoxic effects <span class="citation" data-cites="17k3PVuIy">[<a href="#ref-17k3PVuIy" role="doc-biblioref">18</a>]</span>. Despite this, mobilized colistin resistance (MCR) has emerged in clinical pathogens <span class="citation" data-cites="vQTzfitx">[<a href="#ref-vQTzfitx" role="doc-biblioref">19</a>]</span> likely due to transmission from agricultural usage of colistin <span class="citation" data-cites="11oxu50eO">[<a href="#ref-11oxu50eO" role="doc-biblioref">20</a>]</span>. This further underscores the vital importance of surveillance for evolution and potential transmission of AMR between environmental, agricultural, and clinical samples.</p>
<p>This kind of surveillance requires large-scale phylogenetic analyses which characterise the total diversity, context, and spread of these key resistance determinants. In order to perform these types of analyses we need two resources: a clear consolidated definition of what constitutes an AMR-related gene and a large amount of sequenced microbial genomes. However, naming and defining AMR genes is difficult to a complex legacy of historical nomenclature (prior to widespread DNA sequencing), and a huge quantity of active AMR research from a broad-range of stakeholders including academia, government organizations, human health, animal health, and agri-food industries <span class="citation" data-cites="10jnMhzRm">[<a href="#ref-10jnMhzRm" role="doc-biblioref">6</a>]</span>.</p>
<p>AMR determinants are typically classified into families <span class="citation" data-cites="nvbyXyPe">[<a href="#ref-nvbyXyPe" role="doc-biblioref">21</a>]</span>, some of which are based on phenotypic properties, and others on sequence variation. The nomenclature is usually an acronym with varying representations, such as the mechanism of resistance, the organism it is associated with, or the molecular structre. Each acronym is assigned a numerical value to distinguish variants. Each sequence, determined to be novel by some criterion, is assigned a new number. For example, the MCR phosphoethanolamine transferase gene family inhibits the binding of colistin to the cell membrane. The MCR-3 gene has a sequence identity which has been deemed to be significantly different from that of the other MCR genes in the family. MCR-3.1, has been deemed to be a variant of the MCR-3 gene, which has a slight variation in gene sequence from other MCR-3 genes <span class="citation" data-cites="x2VQlvOO">[<a href="#ref-x2VQlvOO" role="doc-biblioref">22</a>]</span>. Members of the NDM beta-lactamase gene family are named using a different set of criteria. In this case, NDM-1, and NDM-2 differ only by a single amino acid substitution. This system for classification has the potential to be misleading when conducting AMR research. The amount of diversity encompassed within different names and alleles is hugely variable across AMR genes. For example, NDM has little diversity, while MCR has a large amount of sequenced microbial genomes. Some AMR families are not even defined on the basis of sequence space but phenotypically. An example of this is the OXA beta-lactamase family, which is a phenotypically characterized family, with several different homologous sets of genes. OXA-20 has less than 20% amino acid sequence identity when compared with several other OXA-48-like family beta-lactamases <span class="citation" data-cites="1AFSAuSwx">[<a href="#ref-1AFSAuSwx" role="doc-biblioref">23</a>]</span>. This is a problem when attempting to characterize the AMR determinants by sequence similarity.</p>
<p>High-quality manually curated AMR databases such as the Comprehensive Antibiotic Resistance Database (CARD) <span class="citation" data-cites="nvbyXyPe">[<a href="#ref-nvbyXyPe" role="doc-biblioref">21</a>]</span> provide a unified resource for the definition, nomenclature and classification of AMR genes. CARD organizes this information through the Antibiotic Resistance Ontology (ARO), a controlled vocabulary with defined relationships. There are two datasets within CARD, CARD:Canonical and CARD:Resistomes &amp; Variants (R&amp;V). Canonical is a conservative set of AMR genes and mutations that have been experimentally verified as being associated with resistance in a peer-reviewed publication. As a consequence, most of the entries in CARD:Canonical are derived from the most intensively studied organisms, such as the ESKAPE pathogens. On the other hand, CARD:R&amp;V contains AMR sequences that have been detected by searching a set of 88 WHO priority organism genomes from central repositories of publicly available sequencing data such as National Center for Biotechnology Information (NCBI) for sequences similar to corresponding Canonical sequences. This <em>in silico</em> search is performed using CARD’s BLAST/DIAMOND-based Resistance Gene Identifier (RGI) tool and greatly increases the sequence diversity in CARD <span class="citation" data-cites="nvbyXyPe">[<a href="#ref-nvbyXyPe" role="doc-biblioref">21</a>]</span>.</p>
<p>However, this is still limited to a subset of all genomes currently available in central International Nucleotide Sequence Database Collaboration (INSDC) repositories like NCBI. If we want to thoroughly understand the evolution and spread of AMR genes we need to analyse as many genomes as possible. Unfortunately, the genomes in databases are largely sequenced from microbes that can be easily cultured. As only a subset of microbial diversity can be cultured, this means many of our existing genomes aren’t necessarily representative of the environment from which they were sampled <span class="citation" data-cites="wapKbEHA">[<a href="#ref-wapKbEHA" role="doc-biblioref">24</a>]</span>. Recently, techniques have been developed that allow the recovery of genomes from metagenomic data e.g., <span class="citation" data-cites="DTtDOvle">[<a href="#ref-DTtDOvle" role="doc-biblioref">25</a>]</span>. As metagenomic sequencing, the direct sequencing of all DNA in a sample, doesn’t require culturing these metagenome-assembled genomes (MAGs) represent a huge source of novel microbial diversity. Parks et. al. <span class="citation" data-cites="wapKbEHA">[<a href="#ref-wapKbEHA" role="doc-biblioref">24</a>]</span>, generated new 7,903 bacterial and archaeal (Uncultured Bacteria and Archaea; UBA) MAGs representing &gt;30% increase in the sampled phylogenetic diversity of the bacteria and archaea.</p>
<p>By combining this MAG dataset with CARD:R&amp;V, and other NCBI non-redundant data, we are able to greatly increase the breadth of data available for surveillance beyond clinical samples and environments. Phylogenetic analyses of this data can then be used to characterise the evolution and previously unexplored diversity of these critical public-health priority carbapenemase (KPC, NDM, and OXA-48) and colistin AMR gene families. These analyses will also highlight inconsistencies and possible refinements in the classification and nomenclature of these genes and their variant alleles. Therefore, in this work we present an in-depth investigation of the classification and phylogeny of KPC, NDM, OXA-48, and MCR gene families across all currently available sequenced genomes and MAGs.</p>
<h2 id="methods">Methods</h2>
<h3 id="data">Data</h3>
<p>We surveyed four separate databases to assess the diversity of known and candidate AMR genes. These sources include CARD:Canonical and CARD:R&amp;V (version 3.0.5) sequences for the KPC, NDM, OXA-48, and MCR gene families, and associated homologs from NCBI non-redundant data and UBA MAG data. The 32 canonical MCR sequence variants (Table <a href="#tbl:mcr-variants">1</a>), 18 canonical KPC sequence variants (Table <a href="#tbl:kpc-variants">2</a>), 14 canonical NDM sequence variants (Table <a href="#tbl:ndm-variants">3</a>), and 6 canonical OXA-48-like sequence variants (Table <a href="#tbl:oxa-variants">4</a>), as labeled in the CARD database, were obtained as a reference. The NCBI non-redundant data consisted of all non-redundant GenBank, Protein Data Bank (PDB), SwissProt, Protein information resource (PIR), Protein research foundation (PRF) data from May 17, 2019. The MAG data was derived from a data-set of 7,903 draft quality MAGs generated from Sequence Read Archive data by Parks et. al <span class="citation" data-cites="wrBRBdFb">[<a href="#ref-wrBRBdFb" role="doc-biblioref">27</a>]</span>. The SRA metagenomic data from which these MAGs were derived were chosen to represent lineages which were under-sampled (e.g., environmental and non-human gastrointestinal samples).</p>
<h3 id="querying-the-data">Querying the data</h3>
<p>The CARD canonical sequences from each family were used to perform a multiple query BLASTP (version 2.5.0 <span class="citation" data-cites="s9ycaHcq">[<a href="#ref-s9ycaHcq" role="doc-biblioref">28</a>]</span>) against the prevalence BLAST database with an e-value threshold and query coverages shown in Table 1. Many of the sequences are nearly identical, thus they were further processed by clustering with CD-HIT version 4.8.1 at a minimum sequence identity as per Table 1.</p>
<p>The reference CARD canonical sequences were also used to perform a multiple query BLASTP against the NCBI non-redundant database with an e-value and query coverage threshold indicated in Table 1 There are many highly sampled taxa and genes in the non-redundant database. To balance the distribution, and reduce the size of the non-redundant sequence set, CD-HIT version 4.8.1 was also used to cluster the data as per Table 1.</p>
<p>For the metagenomic data, RGI version 5.0 with CARD database version 3.02 <span class="citation" data-cites="1ByMfX8Y1">[<a href="#ref-1ByMfX8Y1" role="doc-biblioref">3</a>]</span> was run on the contigs of the 7903 MAGs with the inclusion of loose, perfect, and strict hits. The search strings in Table S1 were used to extract results from the RGI output. The CARD canonical sequences were used to perform a multiple query BLASTP against this UBA blast database with a e-value threshold and query coverage from Table 1. The minimum BLAST query coverages were chosen to produce meaningful alignments.</p>
<p>For all sequence variants obtained from each data source, redundant results for the prevalence, NCBI, and UBA queries were filtered from these BLASTP results by retrieving only the longest sequence for each unique Subject Taxonomy ID.</p>
<style type="text/css">
.tg  {border-collapse:collapse;border-spacing:0;}
.tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
  overflow:hidden;padding:10px 5px;word-break:normal;}
.tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
  font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
.tg .tg-yj5y{background-color:#efefef;border-color:inherit;text-align:center;vertical-align:top}
.tg .tg-j4pq{background-color:#efefef;border-color:#000000;text-align:center;vertical-align:top}
.tg .tg-llyw{background-color:#c0c0c0;border-color:inherit;text-align:left;vertical-align:top}
.tg .tg-0pky{border-color:inherit;text-align:left;vertical-align:top}
.tg .tg-0lax{text-align:left;vertical-align:top}
</style>
<table class="tg">
<thead>
<tr>
<th class="tg-yj5y" rowspan="2">
AMR<br>Determinant
</th>
<th class="tg-yj5y" colspan="3">
Maximum BLAST<br>e-value<br>
</th>
<th class="tg-yj5y" colspan="3">
Minimum BLAST <br>Query Coverage
</th>
<th class="tg-j4pq" colspan="3">
CD-HIT Sequence <br>Identity Threshold
</th>
</tr>
<tr>
<td class="tg-llyw">
CARD<br>
</td>
<td class="tg-llyw">
NCBI<br>
</td>
<td class="tg-llyw">
UBA<br>
</td>
<td class="tg-llyw">
CARD<br>
</td>
<td class="tg-llyw">
NCBI<br>
</td>
<td class="tg-llyw">
UBA<br>
</td>
<td class="tg-llyw">
CARD<br>
</td>
<td class="tg-llyw">
NCBI<br>
</td>
<td class="tg-llyw">
UBA<br>
</td>
</tr>
</thead>
<tbody>
<tr>
<td class="tg-0pky">
MCR<br>
</td>
<td class="tg-0pky">
1e-160
</td>
<td class="tg-0pky">
1e-160
</td>
<td class="tg-0pky">
1e-160
</td>
<td class="tg-0pky">
98
</td>
<td class="tg-0pky">
98
</td>
<td class="tg-0pky">
98
</td>
<td class="tg-0pky">
100
</td>
<td class="tg-0pky">
100
</td>
<td class="tg-0pky">
100
</td>
</tr>
<tr>
<td class="tg-0pky">
KPC
</td>
<td class="tg-0pky">
1e-100
</td>
<td class="tg-0pky">
1e-40
</td>
<td class="tg-0pky">
1e-10
</td>
<td class="tg-0pky">
99
</td>
<td class="tg-0pky">
99
</td>
<td class="tg-0pky">
60
</td>
<td class="tg-0pky">
99
</td>
<td class="tg-0pky">
70
</td>
<td class="tg-0pky">
100
</td>
</tr>
<tr>
<td class="tg-0pky">
NDM
</td>
<td class="tg-0pky">
1e-160
</td>
<td class="tg-0pky">
1e-160
</td>
<td class="tg-0pky">
1e-10
</td>
<td class="tg-0pky">
98
</td>
<td class="tg-0pky">
98
</td>
<td class="tg-0pky">
60
</td>
<td class="tg-0pky">
100
</td>
<td class="tg-0pky">
100
</td>
<td class="tg-0pky">
100
</td>
</tr>
<tr>
<td class="tg-0pky">
OXA
</td>
<td class="tg-0pky">
1e-100
</td>
<td class="tg-0pky">
1e-100
</td>
<td class="tg-0pky">
1e-10
</td>
<td class="tg-0pky">
90
</td>
<td class="tg-0pky">
99
</td>
<td class="tg-0pky">
95
</td>
<td class="tg-0pky">
N/A
</td>
<td class="tg-0pky">
N/A
</td>
<td class="tg-0pky">
N/A
</td>
</tr>
<tr>
<td class="tg-0lax" colspan="10">
<b>Table 1</b>: Parameters used with BLAST and CD-HIT to obtain sequences for the phylogenies<br>of MCR phosphoethanolamine transferase, and KPC, NDM, and OXA-48 beta-lactamases.<br>For each determinant, the minimum BLAST e-values, minimum query coverages, and CD-HIT Sequence Identity <br>threshold for clustering sequences are displayed for each database.<br>CARD represents the RGI-CARD prevalence sequence data, NCBI represents the NCBI non-redundant sequence<br>data, and UBA represents the MAG data source.<br><br>After filtering, OXA did not have an overabundance of close sequences and was not clustered (marked as N/A)<br>
</td>
</tr>
</tbody>
</table>
<p>Table 1: Names of the canonical MCR phosphoethanolamine transferase variants from CARD database {#tbl:params}</p>
<h3 id="sequence-alignment">Sequence Alignment</h3>
<p>Two alignments were created for each AMR determinant under study. The first alignment was created to compare the phylogenetic relationship of the canonical and prevalence (putaive) sequences. The second alignment was created for an comparison of sequences from all four databases.</p>
<p>The putative AMR alignment was made up of the sequences from the CARD prevalence data were combined in one multi-FASTA format file with the canonical sequences and an outgroup chosen for each AMR family as per table ?? [TODO: make a table of outgroups in supplemental]. In the second, more diverse alignment, The filtered sequences from NCBI non-redundant data, CARD prevalence data, UBA data were all combined into one multi-FASTA format file with the canonical sequences with the same outgroup sequences (table ??).</p>
<p>This set of combined amino acid sequences were aligned with MAFFT-LINSI <span class="citation" data-cites="1DMuBNrog">[<a href="#ref-1DMuBNrog" role="doc-biblioref">29</a>]</span> version 7.40 and trimmed by trimal <span class="citation" data-cites="GCsU1nyf">[<a href="#ref-GCsU1nyf" role="doc-biblioref">30</a>]</span> version 1.4.rev22 using the automated1 option.</p>
<h3 id="creation-of-phylogenies">Creation of phylogenies</h3>
<p>For each alignment under each AMR family under study, IQ-TREE multicore version 1.6.9 <span class="citation" data-cites="JOWSuu8G">[<a href="#ref-JOWSuu8G" role="doc-biblioref">31</a>]</span> was used to build a bootstrapped tree with -bb 1000 with the G+I+G4 model of substitution. Tree visualizations were created with ETE Toolkit <span class="citation" data-cites="5F7ii9Ji">[<a href="#ref-5F7ii9Ji" role="doc-biblioref">32</a>]</span> version 3. The trees were annotated with taxonomic information for each rank by using ETE’s NCBI taxonomy module to translate the names result set from the various databases to taxonomic identifiers. The taxonomic identifier was used to query NCBI’s taxonomy database from May 17, 2019, for the taxonomy information. In cases where there were multiple hits for genes, the most recent common ancestor was used to query the taxonomic information, therefore some leaves in the trees were annotated with only partial taxonomic information.</p>
<h2 id="results">Results</h2>
<h3 id="phylogenetic-analysis-of-mcr-sequences">Phylogenetic analysis of MCR sequences</h3>
<p>A phylogeny with only CARD:Canonical and CARD:R&amp;V MCR gene family sequences was created, showing the relationships without diversity from NCBI or the MAGs, including an out-group, 32 CARD:Canonical sequences, and 54 CARD:R&amp;V sequences. The resulting tree in Figure {<span class="citation" data-cites="fig:canon-prev-tree">[<span class="citeproc-not-found" data-reference-id="fig:canon-prev-tree"><strong>???</strong></span>]</span>} shows MCR family members 1 through 9 forming distinct sub-clades in the tree. MCR-1, MCR-2, and MCR-6 form a closely related clade. This clade is closely related to ICR-Mc <span class="citation" data-cites="JT9eRkR8">[<a href="#ref-JT9eRkR8" role="doc-biblioref">33</a>]</span>, another phosphoethanolamine transferase which confers colistin resistance. MCR-3 and MCR-7, also form a closely related clade.</p>
<p><img src="images/mcr_canon_prev_full.mintax.svg" alt="Phylogenetic relationship of 32 canonical (labels prefixed with ‘lcl_canon’ in yellow), 54 CARD:R&amp;V (labels prefixed with ‘lcl_prev’ in tan) MCR family sequences, and an outgroup from Betaproteobacteria (‘lcl_outgroup’ in grey). Each MCR variant is coloured based on its allele number." id="fig:canon-prev-tree" /> The tree visualisation was further condensed to only a single representative for each MCR allele (figure <a href="#fig:canon-prev-tree-collapsed">1</a>). Regions of the tree vary in their potential to accept additional diversity. Some clades are tightly related, while others have more distance between them. Newly discovered genes could some day be found to be sister to these more distantly related clades. Regions containing more evolutionary distance are accommodating to additional diversity, while closely related regions are not.</p>
<figure>
<img src="images/mcr_canon_prev_collapsed.mintax.svg" alt="Figure 1: Phylogenetic relationship of 9 representative MCR family sequences, and an outgroup from Betaproteobacteria. (labels prefixed with ‘lcl_canon’ in yellow are canonical, labels prefixed with ‘lcl_prev’ in tan are CARD:R&amp;V, and an outgroup from Betaproteobacteria is prefixed with ‘lcl_outgroup’ in grey)" id="fig:canon-prev-tree-collapsed" /><figcaption><span>Figure 1:</span> Phylogenetic relationship of 9 representative MCR family sequences, and an outgroup from Betaproteobacteria. (labels prefixed with ‘lcl_canon’ in yellow are canonical, labels prefixed with ‘lcl_prev’ in tan are CARD:R&amp;V, and an outgroup from Betaproteobacteria is prefixed with ‘lcl_outgroup’ in grey)</figcaption>
</figure>
<p>The phylogenies represented by Figures <a href="#fig:mcr-3-9">2</a>, <a href="#fig:mcr-5-icr">3</a>, and <a href="#fig:canon-prev-nrdb-uba-tree">7</a> were created with additional diversity by querying the NCBI non-redundant database, and the MAGs from the Parks <em>et. al.</em> UBA data <span class="citation" data-cites="wrBRBdFb">[<a href="#ref-wrBRBdFb" role="doc-biblioref">27</a>]</span>. The filtering criteria in Table 1 were applied to these data sets.</p>
<p>After filtering, 409 NCBI sequence hits, labeled as phosphoethanolamine lipid A transferase, and 104 NCBI sequence hits, labeled as MCR family, were selected for addition to the phylogeny. The 7903 draft quality MAGs were queried for AMR genes with RGI. RGI produced 1457246 loose hits, 7171 strict hits, and 310 perfect hits. After applying the filters from Table 1, only 63 MAG derived genes were retained for the phylogeny. Due to the large size, the phylogeny displayed in Figure <a href="#fig:canon-prev-nrdb-uba-tree">7</a> was subdivided into Figures <a href="#fig:mcr-3-9">2</a>, and <a href="#fig:mcr-5-icr">3</a>. The subdivisions were chosen based on a pruning at the point of the most recent common ancestor of sets of canonical MCR labeled leaves. The set including MCR-3, MCR-7, and MCR-9, and the set including MCR-1, MCR-2, MCR-5, MCR-6, and ICR-Mc, contained several clades sister to some of the canonical and CARD:R&amp;V leaves. Other choices for leaf sets contained added diversity that was distant from canonical sequences, and were not chosen as a focused figures.</p>
<p>Sister to the canonical MCR-3 sequences, and the most recent common ancestor of MCR-3 and MCR-7 clades (figure <a href="#fig:mcr-3-9">2</a>), there is a clade of sequences from NCBI which have been reported as MCR-3. Present within this clade is a single result from the Parks <em>et. al.</em> UBA dataset <span class="citation" data-cites="wrBRBdFb">[<a href="#ref-wrBRBdFb" role="doc-biblioref">27</a>]</span>, UBA705. This leaf is labelled as <em>Aeromonas hydrophila</em>, and is associated with rice paddy field-soil microbial fuel cells. This is present within the clade alone with several other canonical and non-redundant <em>Aeromonas</em>. <em>A. hydrophila</em> is a species which has been found to have an MCR-3 gene.</p>
<p>Sister to the clade containing MCR 7, and the most recent common ancestor of MCR 3 and MCR 7 clades, is a clade of NCBI sourced phosphoethanolamine lipid A transferase. This clade consists of the genus of Gram-negative bacteria, <em>Aeromonas</em>, <span class="citation" data-cites="16EwPCiJT">[<a href="#ref-16EwPCiJT" role="doc-biblioref">34</a>]</span> which is sometimes involved in human infection. This clade also includes an <em>A. veronii</em> hit from epidermal mucus of <em>Anguilla anguilla</em> in the UBA data. Sister to the MCR 9 clade and the most recent common ancestor of MCR 3 and 9, a clade of <em>Aeromonas</em> associated phosphoethanolamine lipid A transferases appear. The MCR 9 containing clade contains 8 UBAs and several NCBI non-redundant hits not reported as MCR family genes. 5 Loose hits for MCR in a <em>Leclercia adecarboxylata</em> branch within this clade. These <em>Leclercia</em> branch as a sister to the common ancestor of MCR-9 and MCR-3 which are well supported. The <em>Leclercia</em> UBAs were all sampled from New York City MTA subway. <em>Acinetobacter</em> is another opportunistic pathogen <span class="citation" data-cites="8uz5m0fP">[<a href="#ref-8uz5m0fP" role="doc-biblioref">35</a>]</span> which is becoming resistant to many antimicrobials.</p>
<figure>
<img src="images/mcr3_mcr9_diversity.mintax.svg" alt="Figure 2: Clade containing putative MCR3 and MCR 9 clades pruned from phylogenetic relationship of 32 canonical (labels prefixed with ‘lcl_canon’_ in yellow), and 54 CARD:R&amp;V (labels prefixed with ‘lcl_prev’ in tan) MCR family sequences, 595 NCBI non-redundant sequences (labels prefixed with ‘lcl_nrdb’ in blue), and 91 UBA sourced sequences (labels prefixed with ‘lcl_uba’ in green). Each MCR variant is coloured based on its allele number. If the sequence is not reported to be MCR family it is coloured in grey." id="fig:mcr-3-9" /><figcaption><span>Figure 2:</span> Clade containing putative MCR3 and MCR 9 clades pruned from phylogenetic relationship of 32 canonical (labels prefixed with ‘lcl_canon’_ in yellow), and 54 CARD:R&amp;V (labels prefixed with ‘lcl_prev’ in tan) MCR family sequences, 595 NCBI non-redundant sequences (labels prefixed with ‘lcl_nrdb’ in blue), and 91 UBA sourced sequences (labels prefixed with ‘lcl_uba’ in green). Each MCR variant is coloured based on its allele number. If the sequence is not reported to be MCR family it is coloured in grey.</figcaption>
</figure>
<p>Sister to the MCR 5 and ICR-Mc clades (figure {#fig:mcr-5mayb-icr}), there appear clades of diversity in the genus <em>Psychrobacter</em>. A descendant to this ancestor, is a clade of <em>Psycrobacter</em> species bacteria.</p>
<p>This clade includes several hits from the non-redundant database, and two hits from the UBA data. According to the UBA <span class="citation" data-cites="wrBRBdFb">[<a href="#ref-wrBRBdFb" role="doc-biblioref">27</a>]</span> data, the identity of these samples are, UBA3068, A <em>Psychrobacter sp.</em>, sampled from Oil polluted marine microbial communities and, UBA4193, a <em>Psychrobacter sp.</em>, sampled from the New York City subway. The quality information for these sequences, shown in table ?? [TODO: do we need a table with quality information], shows that UBA3068 is near complete, while UBA4193 is only partial. It is encouraging to see that even with the quality difference, these two sequences branch in the expected clade. <em>Psychrobacter</em> <span class="citation" data-cites="BjoH1Vii">[<a href="#ref-BjoH1Vii" role="doc-biblioref">36</a>]</span> is a widespread genus and includes many cold adapted bacteria, it is an opportunistic pathogen, and has been found to sometimes be a cause of infections in humans, animals, and fish. Many new species of <em>Psychrobacter</em> have been discovered in cold climates <span class="citation" data-cites="WLzELTTa">[<a href="#ref-WLzELTTa" role="doc-biblioref">37</a>]</span>. Some of the species have been shown to be resistant to colistin, like <em>P. vallis ps. nov.</em> and <em>P. aquaticus ps. nov.</em> <span class="citation" data-cites="2NZAmp2H">[<a href="#ref-2NZAmp2H" role="doc-biblioref">38</a>]</span> and is sister to [TODO: display new tree such that <em>Acinetobacter</em> is shown] Another clade sister to hese two variants contains phosphoethanolamine transferases from the genus <em>Stenotrophomonas</em>, including <em>S. maltophilia</em> and <em>S. acidaminiphila</em> which is a multidrug-resistant opportunistic pathogen <span class="citation" data-cites="Ovca71PG">[<a href="#ref-Ovca71PG" role="doc-biblioref">39</a>]</span>. Several UBA hits for <em>S. maltophilia</em> show up as shotgun sequencing of environmental samples. [TODO: quality information]</p>
<figure>
<img src="images/mcr5_icrmc_diversity.mintax.svg" alt="Figure 3: Clade containing putative MCR 5 and ICR-Mc clades pruned from phylogenetic relationship of 32 canonical labels (prefixed with ‘lcl_canon’_ in yellow), and 54 CARD:R&amp;V(labels prefixed with ‘lcl_prev’ in tan) MCR family sequences, 595 NCBI non-redundant sequences (labels prefixed with ‘lcl_nrdb’ in blue), and 91 UBA sourced sequences (labels prefixed with ‘lcl_uba’ in green). Each MCR variant is coloured based on its allele number. If the sequence is not reported to be MCR family it is coloured in grey." id="fig:mcr-5-icr" /><figcaption><span>Figure 3:</span> Clade containing putative MCR 5 and ICR-Mc clades pruned from phylogenetic relationship of 32 canonical labels (prefixed with ‘lcl_canon’_ in yellow), and 54 CARD:R&amp;V(labels prefixed with ‘lcl_prev’ in tan) MCR family sequences, 595 NCBI non-redundant sequences (labels prefixed with ‘lcl_nrdb’ in blue), and 91 UBA sourced sequences (labels prefixed with ‘lcl_uba’ in green). Each MCR variant is coloured based on its allele number. If the sequence is not reported to be MCR family it is coloured in grey.</figcaption>
</figure>
<h3 id="phylogenetic-analysis-of-kpc">Phylogenetic analysis of KPC</h3>
<p>Using the 18 canonical sequences from Table <a href="#tbl:kpc-variants">2</a> as query sequences, 25 CARD:R&amp;V sequences, 376 non-redundant sequences, and 6 UBA sequences were recovered from the various databases. The result was combined in the phylogeny displayed in figure <a href="#fig:kpc-tree">11</a>. This phylogeny shows that many of the resulting sequences are distant from the canonical sequences. The main clade in figure <a href="#fig:kpc-main-tree">4</a> shows that many of the AMR determinants are named as distinct sequences, while not having much phylogenetic distance between them. The most closely related clade is a clade containing IMI and SME-type beta-lactamases. The remaining sequences are in a clade containing a large amount of class A beta-lactamases from the non-redundant data. All of the UBA sequences were present in this portion of the tree.</p>
<figure>
<img src="images/kpc_main_clade.mintax.svg" alt="Figure 4: A clade pruned from the phylogeny in figure {#fig:kpc-tree} which represents the most closely related genes to the KPC family genes. 18 canonical labels (prefixed with ‘lcl_canon’_ in yellow), and 25 CARD:R&amp;V (labels prefixed with ‘lcl_prev’ in tan), and 28 NCBI non-redundant sequences (labels prefixed with ‘lcl_nrdb’ in blue). Each KPC variant is coloured based on its allele number. If the sequence is not reported to be KPC family it is coloured in grey." id="fig:kpc-main-tree" /><figcaption><span>Figure 4:</span> A clade pruned from the phylogeny in figure {#fig:kpc-tree} which represents the most closely related genes to the KPC family genes. 18 canonical labels (prefixed with ‘lcl_canon’_ in yellow), and 25 CARD:R&amp;V (labels prefixed with ‘lcl_prev’ in tan), and 28 NCBI non-redundant sequences (labels prefixed with ‘lcl_nrdb’ in blue). Each KPC variant is coloured based on its allele number. If the sequence is not reported to be KPC family it is coloured in grey.</figcaption>
</figure>
<h3 id="phylogenetic-analysis-of-ndm">Phylogenetic analysis of NDM</h3>
<p>14 canonical sequences were used to query the data for NDM beta-lactamase. The genes retained for the phylogeny were 14 canonical sequences, 8 CARD:R&amp;V sequences, 12 non-redundant sequences, 1 UBA sequence, and an out-group. This resulted in the phylogenetic relationship in Figure <a href="#fig:ndm-tree">10</a>.</p>
<figure>
<img src="images/ndm_full.mintax.svg" alt="Figure 5: Phylogenetic relationship of The NDM genes with 14 canonical (labels prefixed with ‘lcl_canon’ in yellow), 8 CARD:R&amp;V (labels prefixed with ‘lcl_prev’ in tan), 12 NCBI non-redundant database sequences (labeled as ‘lcl_nrdb’ in blue), 1 UBA sequence (labeled as ‘lcl_uba’ in green), and an outgroup from Betaproteobacteria (labeled as ‘lcl_outgroup’ in grey). Each NDM variant is coloured based on its allele number." id="fig:ndm-tree-all" /><figcaption><span>Figure 5:</span> Phylogenetic relationship of The NDM genes with 14 canonical (labels prefixed with ‘lcl_canon’ in yellow), 8 CARD:R&amp;V (labels prefixed with ‘lcl_prev’ in tan), 12 NCBI non-redundant database sequences (labeled as ‘lcl_nrdb’ in blue), 1 UBA sequence (labeled as ‘lcl_uba’ in green), and an outgroup from Betaproteobacteria (labeled as ‘lcl_outgroup’ in grey). Each NDM variant is coloured based on its allele number.</figcaption>
</figure>
<p>The 12 sequences resulting from the NCBI data included 3 genes reported to be NDM-1, NDM-3, and NDM-5, while the other 4 genes were reported to be general beta-lactamase/carbapenemase hits. As in the KPC phylogeny (figure <a href="#fig:kpc-main-tree">4</a>), there is very little phylogenetic difference between the named variants of NDM. The only phylogenetically distinct genes in this clade were the CARD:R&amp;V hit for NDM-1, and the NCBI hits labeled as NDM-1 and NDM-5. The naming of these sequences seem not to be congruent with the phylogenetic relationships present in the tree. Some of the NDM-1 and NDM-5 genes are more closely related to the other NDM variants than themselves. One BLAST result from the UBA data branches far from the clade containing the canonical sequences. This indicates that under the coverage queried, there are no reasonably detectable NDM homologues in the UBA data. The alignment of this UBA under this relaxed query coverage of 60% (see Table 1) is questionable. Reducing query coverage further could produce more sequences from the UBA data, but the resulting alignment would produce a meaningless phylogeny.</p>
<h3 id="phylogenetic-analysis-of-oxa-48">Phylogenetic analysis of OXA-48</h3>
<p>In querying for the OXA-48 portion of the analysis, multiple BLAST results for UBA sequences were found, but the hits were too phylogenetically dissimilar to draw a conjecture about their relationship to the OXA family. 6 canonical sequences, and the 14 CARD:R&amp;V sequences, and 20 non-redundant sequences are combined in the phylogeny in Figure <a href="#fig:oxa-tree">8</a></p>
<figure>
<img src="images/oxa_main_clade.mintax.svg" alt="Figure 6: The main phylogenetic relationship pruned from the tree inferred from OXA-48-like sequences from figure {#fig:oxa-tree}. 6 canonical (labels prefixed with ‘lcl_canon’_ in yellow), and 14 CARD:R&amp;V (labels prefixed with ‘lcl_prev’ in tan), 20 NCBI non-redundant sequences (labels prefixed with ‘lcl_nrdb’ in blue). Each OXA-48 variant is coloured based on its allele number. If the sequence is not reported to be OXA-48 family it is coloured in grey." id="fig:oxa-main-tree" /><figcaption><span>Figure 6:</span> The main phylogenetic relationship pruned from the tree inferred from OXA-48-like sequences from figure {#fig:oxa-tree}. 6 canonical (labels prefixed with ‘lcl_canon’_ in yellow), and 14 CARD:R&amp;V (labels prefixed with ‘lcl_prev’ in tan), 20 NCBI non-redundant sequences (labels prefixed with ‘lcl_nrdb’ in blue). Each OXA-48 variant is coloured based on its allele number. If the sequence is not reported to be OXA-48 family it is coloured in grey.</figcaption>
</figure>
<p>The OXA-48 CARD:R&amp;V hits added further diversity to the reference OXA-48-like sequences. OXA-436 <span class="citation" data-cites="KEhNzdys">[<a href="#ref-KEhNzdys" role="doc-biblioref">40</a>]</span> was found, clustered with no other gene, in the CARD:R&amp;V data, and OXA-514 and OXA-515 were found in the non-redundant data. The named variants are too similar to produce significant phylogenetic resolution.</p>
<h2 id="discussion">Discussion</h2>
<h3 id="an-expanded-set-of-candidate-resistance-genes">An expanded set of candidate resistance genes</h3>
<p>Homologous sequences from NCBI, CARD:R&amp;V data, and a large set of MAGs, were queried, filtered, and added to phylogenies. The filtering criteria were chosen to ignore small changes between sequences (99-100% sequence identity threshold), and to produce meaningful alignments in the final phylogenies (90-99% query coverage).</p>
<p>NCBI and CARD:R&amp;V added new homologues to the phylogenies, however, these conservative filtering criteria contributed obvious novel homologues from only the MCR MAG analysis, and not in the three beta lactamase MAG analyses. This difference could be explained by plasmid distributions. Plasmids could have been poorly recovered in the assembly phase of the SRA experiments, many of which produce short reads (Illumina HiSeq 2000 and 2500). TODO: Look at The number/percentages of completely sequenced genomes, completely sequenced plasmids analyzed for each pathogen, and see if there is any concrete evidence that the above is actually happening. This seems to be problematic with bias in sampling, so other ideas/sources that may be promising are welcome. Plasmids from short reads are known to be difficult to assemble<span class="citation" data-cites="12zFifp5x">[<a href="#ref-12zFifp5x" role="doc-biblioref">41</a>]</span> <span class="citation" data-cites="12zFifp5x">[<a href="#ref-12zFifp5x" role="doc-biblioref">41</a>]</span> <span class="citation" data-cites="DTtDOvle">[<a href="#ref-DTtDOvle" role="doc-biblioref">25</a>]</span>.</p>
<h3 id="phylogenetic-positioning-of-amr-genes-from-ncbi-and-mags">Phylogenetic positioning of AMR genes from NCBI and MAGs</h3>
<p>Another way in which the analyses of MCR and the beta lactamases differ is in the spread and diversity of the relationships. In addition to MAG sources, which are completely absent from the beta lactamase trees, the MCR tree contains several sequences from NCBI’s non-redundant database which land close to canonical clades. Some of the added NCBI and MAG sourced sequences are not so close, but share ancestors with canonical sequences nonetheless. This suggests that there could be more diversity to be found within the MCR class. In contrast, the added diversity in phylogenies from OXA-48, KPC is distant from the canonical and CARD:R&amp;V sequences. The matches which are close, are nearly identical to the canonical and CARD:R&amp;V sequences. The beta-lactamase phylogenies are low in gene diversity, and are more tightly defined than the MCR phylogenies.</p>
<h3 id="phylogenetic-relationships-vs.-nomenclature">Phylogenetic relationships vs. nomenclature</h3>
<p>In assessing the value of the relationships present in these more tightly related phylogenies, it is surprising that the literature has produced so many named variants. In the tree for NDM, we have named variants numbered from NDM 1 to NDM 14, yet most of the tree would be more appropriately displayed with multifurcating nodes. There are fifteen named variants, but there are only three weakly distinct clades, and little bootstrap support (TODO: look at SNPs put this in results?). The most divergent NDM sequence is a CARD:R&amp;V sequence which is labelled as NDM-1, yet is more distant from other homologs named NDM-1 than other NDM-1 genes. The criteria for naming these genes are seemingly arbitrary. It is also possible that incentive to find new names for producing publication contributes to many of the more redundant variants. Similarly, KPC is very tightly related and some of the same arguments could be made for classification of these sequences. In the tree for OXA-48 like genes, we see more diversity, with branches of more support. The tree forms few tightly related clades, and allows some diversity from NCBI to be added to the tree. The OXA group, as a whole, is very diverse, but only because they are classified phenotypically. If the OXA were more thoroughly separated into actual homologous groups, perhaps we would see more diversity added to the group, and perhaps even from metagenomic data. These groups of OXA could behave very differently and some could have data recovery similar to that of MCR, but we are missing out on this because of the phenotypic naming scheme over a rigorous scheme which takes into account the genetic similarity. In MCR, the naming situation is less arbitrary. The closely related variants are classified under the same primary number. Several versions of MCR-1 exist, for example, and all form a clade.</p>
<h3 id="should-naming-reflect-phylogenetic-relationships">Should naming reflect phylogenetic relationships?</h3>
<p>Finding an ideal nomenclature scheme which within, or generalizing across AMR families is problematic. Nomenclature which has been shown to be phylogenetically consistent, such as that of MCR, is nonetheless subject to ambiguity. Researchers have suggested, for example for MCR <span class="citation" data-cites="x2VQlvOO">[<a href="#ref-x2VQlvOO" role="doc-biblioref">22</a>]</span>, using amino acid sequence identity differences rather than possibly trivial nucleotide differences, but this still subject to ambiguity in cutoff of percent identity. Fixed cutoffs of 2% have been suggested by some researchers<span class="citation" data-cites="eGJmNg9p">[<a href="#ref-eGJmNg9p" role="doc-biblioref">42</a>]</span>, but depending on the mechanism of resistance, the level of diversity required to induce differences in substrate specificity will change between AMR genes. One could argue that the genomic and evolutionary context should be also considered in the nomenclature <span class="citation" data-cites="EhIOtkvv">[<a href="#ref-EhIOtkvv" role="doc-biblioref">43</a>]</span>. It has been suggested, for example that mobilized, and chromosomal genes differ in their evolutionary variance <span class="citation" data-cites="EhIOtkvv">[<a href="#ref-EhIOtkvv" role="doc-biblioref">43</a>]</span>. In addition, the current study demonstrates that the diversity and spread varies between gene families. This could be explained by the way MCR works by modifying the target of polymyxin, unlike the beta-lactamases, which work by destroying the beta-lactams. Justifying naming of a new NDM gene, when all genes available to the public are essentially identical, seems to be problematic, however, further investigations into certain patterns of changes in the amino acid sequences may further granularize our classification. Ultimately, many interesting questions about the nomenclature of these genes are rooted in ontology. Classifying these genes is of utmost importance for communication between scientists, and a naming scheme which seeks a balance between practicality, evolutionary relationships, function, and context, must be found.</p>
<h1 id="supplemental">Supplemental</h1>
<a name="tbl:mcr-variants"></a>
<table>
<caption><span>Table 1:</span> Names of the canonical MCR phosphoethanolamine transferase variants from CARD database </caption>
<colgroup>
<col style="width: 100%" />
</colgroup>
<thead>
<tr class="header">
<th>MCR-9 MCR-1.8 MCR-1.11 MCR-3.4 MCR-3.11 MCR-6.1 MCR-3.3 MCR-2.2 MCR-1.5 MCR-3.6 MCR-3.10 MCR-3.5 MCR-1.10 MCR-1.9 MCR-3.9 MCR-3.8 MCR-3.7 MCR-3.2 MCR-1.6 MCR-1.2 MCR-1.3 MCR-1.4 MCR-8 MCR-7.1 MCR-1.7 MCR-1.12 MCR-1.13 MCR-3.12 MCR-4.2 MCR-4.3 MCR-4.4 MCR-4.5 MCR-5.2</th>
</tr>
</thead>
<tbody>
</tbody>
</table>
<a name="tbl:kpc-variants"></a>
<table>
<caption><span>Table 2:</span> Names of the canonical beta-lactamase variants from CARD database </caption>
<colgroup>
<col style="width: 100%" />
</colgroup>
<thead>
<tr class="header">
<th>KPC-1 KPC-3 KPC-4 KPC-5 KPC-6 KPC-7 KPC-8 KPC-9 KPC-10 KPC-11 KPC-12 KPC-13 KPC-14 KPC-15 KPC-16 KPC-17 KPC-19 KPC-22 KPC-24</th>
</tr>
</thead>
<tbody>
</tbody>
</table>
<a name="tbl:ndm-variants"></a>
<table>
<caption><span>Table 3:</span> Names of the canonical NDM beta-lactamase variants from CARD database </caption>
<colgroup>
<col style="width: 100%" />
</colgroup>
<thead>
<tr class="header">
<th>NDM-1 NDM-2 NDM-5 NDM-3 NDM-4 NDM-6 NDM-7 NDM-8 NDM-9 NDM-10 NDM-11 NDM-12 NDM-13 NDM-14 NDM-17</th>
</tr>
</thead>
<tbody>
</tbody>
</table>
<a name="tbl:oxa-variants"></a>
<table>
<caption><span>Table 4:</span> Names of the canonical OXA beta-lactamase variants from CARD database </caption>
<thead>
<tr class="header">
<th>OXA-162 OXA-163 OXA-181 OXA-204 OXA-232 OXA-48</th>
</tr>
</thead>
<tbody>
</tbody>
</table>
<p>TODO: Quality/CHECKM information for UBAs</p>
<a name="tbl:outgroups"></a>
<table>
<caption><span>Table 5:</span> Outgroups chosen in building the phylogenies of the 4 AMR families. </caption>
<colgroup>
<col style="width: 20%" />
<col style="width: 39%" />
<col style="width: 20%" />
<col style="width: 18%" />
</colgroup>
<thead>
<tr class="header">
<th>Outgroup to AMR Family</th>
<th>Gene</th>
<th>Species</th>
<th>Class</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>MCR</td>
<td>phosphoethanolamine–lipid A transferase</td>
<td>Oxalobacter formigenes</td>
<td>Betaproteobacteria</td>
</tr>
<tr class="even">
<td>KPC</td>
<td>class A beta-lactamase</td>
<td>Mesorhizobium loti</td>
<td>Alpha Proteobacteria</td>
</tr>
<tr class="odd">
<td>NDM</td>
<td>VIM-4 metallo-beta-lactamase</td>
<td>Alcaligenes faecalis</td>
<td>Betaproteobacteria</td>
</tr>
<tr class="even">
<td>OXA-48</td>
<td>OXA-10 family class D beta-lactamase OXA-454</td>
<td>Delftia acidovorans</td>
<td>Betaproteobacteria</td>
</tr>
</tbody>
</table>
<h4 id="mcr-alignment">MCR Alignment</h4>
<p>[TODO: visualization with alignment/entropy] <img src="images/aln_mcr.png" alt="placeholder" id="fig:ndm-tree" /></p>
<h4 id="kpc-alignment">KPC Alignment</h4>
<p>[TODO: visualization with alignment/entropy] <img src="images/aln_kpc.png" alt="placeholder" id="fig:ndm-tree" /></p>
<h4 id="ndm-alignment">NDM Alignment</h4>
<p>[TODO: visualization with alignment/entropy] <img src="images/aln_ndm.png" alt="placeholder" id="fig:ndm-tree" /></p>
<h4 id="oxa-48-alignment">OXA-48 Alignment</h4>
<p>[TODO: visualization with alignment/entropy] <img src="images/aln_oxa.png" alt="placeholder" id="fig:ndm-tree" /></p>
<figure>
<img src="images/mcr191004_canon_prev_nrdb_uba.fasta.aln.trim.treefile.expanded.svg" alt="Figure 7: Phylogenetic relationship of 32 canonical (labels prefixed with lcl_canon_ in yellow), and 54 prevalence (labels prefixed with lcl_prev_ in tan) MCR family sequences, 595 NCBI non-redundant sequences (labels prefixed with lcl_prev_ in blue), and 91 UBA sourced sequences (labels prefixed with lcl_prev_ in green). Each MCR variant is coloured based on its primary numerical value. If the sequence is not reported to be MCR family it is coloured in grey." id="fig:canon-prev-nrdb-uba-tree" /><figcaption><span>Figure 7:</span> Phylogenetic relationship of 32 canonical (labels prefixed with lcl_canon_ in yellow), and 54 prevalence (labels prefixed with lcl_prev_ in tan) MCR family sequences, 595 NCBI non-redundant sequences (labels prefixed with lcl_prev_ in blue), and 91 UBA sourced sequences (labels prefixed with lcl_prev_ in green). Each MCR variant is coloured based on its primary numerical value. If the sequence is not reported to be MCR family it is coloured in grey.</figcaption>
</figure>
<figure>
<img src="images/oxa191128.canon.svg" alt="Figure 8: Phylogenetic relationship (lcl_canon) OXA-48 family sequences along with 14 prevalence (lcl_prev) sequences." id="fig:oxa-tree" /><figcaption><span>Figure 8:</span> Phylogenetic relationship (lcl_canon) OXA-48 family sequences along with 14 prevalence (lcl_prev) sequences.</figcaption>
</figure>
<figure>
<img src="images/oxa191128.expanded.svg" alt="Figure 9: Phylogenetic relationship (lcl_canon) OXA-48 family sequences along with 14 prevalence (lcl_prev) sequences." id="fig:oxa-tree-all" /><figcaption><span>Figure 9:</span> Phylogenetic relationship (lcl_canon) OXA-48 family sequences along with 14 prevalence (lcl_prev) sequences.</figcaption>
</figure>
<figure>
<img src="images/ndm191128.canon.svg" alt="Figure 10: Phylogenetic relationship (lcl_canon) NDM family sequences along with 14 prevalence (lcl_prev) sequences." id="fig:ndm-tree" /><figcaption><span>Figure 10:</span> Phylogenetic relationship (lcl_canon) NDM family sequences along with 14 prevalence (lcl_prev) sequences.</figcaption>
</figure>
<figure>
<img src="images/kpc191127.canon.svg" alt="Figure 11: Phylogenetic relationship (lcl_canon) KPC family sequences along with 14 prevalence (lcl_prev) sequences." id="fig:kpc-tree" /><figcaption><span>Figure 11:</span> Phylogenetic relationship (lcl_canon) KPC family sequences along with 14 prevalence (lcl_prev) sequences.</figcaption>
</figure>
<figure>
<img src="images/kpc191127_canon_prev_nrdb_uba.fasta.aln.trim.treefile.expanded.svg" alt="Figure 12: Phylogenetic relationship (lcl_canon) KPC family sequences along with 14 prevalence (lcl_prev) sequences." id="fig:kpc-tree-all" /><figcaption><span>Figure 12:</span> Phylogenetic relationship (lcl_canon) KPC family sequences along with 14 prevalence (lcl_prev) sequences.</figcaption>
</figure>
<h2 id="references" class="page_break_before">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" role="doc-bibliography">
<div id="ref-Zvf1VhKR">
<p>1. <strong>Antibiotic Resistance☆</strong><br />
B. Périchon, P. Courvalin, C. W. Stratton<br />
<em>Reference Module in Biomedical Sciences</em> (2015) <a href="https://doi.org/dbhq">https://doi.org/dbhq</a><br />
DOI: <a href="https://doi.org/10.1016/b978-0-12-801238-3.02385-0">10.1016/b978-0-12-801238-3.02385-0</a></p>
</div>
<div id="ref-2xaXclNM">
<p>2. <strong>Antibiotic resistance is ancient</strong><br />
Vanessa M. D’Costa, Christine E. King, Lindsay Kalan, Mariya Morar, Wilson W. L. Sung, Carsten Schwarz, Duane Froese, Grant Zazula, Fabrice Calmels, Regis Debruyne, … Gerard D. Wright<br />
<em>Nature</em> (2011-08-31) <a href="https://doi.org/b3wbvx">https://doi.org/b3wbvx</a><br />
DOI: <a href="https://doi.org/10.1038/nature10388">10.1038/nature10388</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21881561">21881561</a></p>
</div>
<div id="ref-1ByMfX8Y1">
<p>3. <strong>CARD 2017: expansion and model-centric curation of the comprehensive antibiotic resistance database</strong><br />
Baofeng Jia, Amogelang R. Raphenya, Brian Alcock, Nicholas Waglechner, Peiyao Guo, Kara K. Tsang, Briony A. Lago, Biren M. Dave, Sheldon Pereira, Arjun N. Sharma, … Andrew G. McArthur<br />
<em>Nucleic Acids Research</em> (2016-10-26) <a href="https://doi.org/f9wbjs">https://doi.org/f9wbjs</a><br />
DOI: <a href="https://doi.org/10.1093/nar/gkw1004">10.1093/nar/gkw1004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27789705">27789705</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5210516">PMC5210516</a></p>
</div>
<div id="ref-kDtOJsGI">
<p>4. <strong>Horizontal transfer of antibiotic resistance genes in clinical environments</strong><br />
Nicole A. Lerminiaux, Andrew D. S. Cameron<br />
<em>Canadian Journal of Microbiology</em> (2019-01) <a href="https://doi.org/gfnrkj">https://doi.org/gfnrkj</a><br />
DOI: <a href="https://doi.org/10.1139/cjm-2018-0275">10.1139/cjm-2018-0275</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30248271">30248271</a></p>
</div>
<div id="ref-OLzrpsK4">
<p>5. <strong>Carbapenem Resistance: A Review</strong><br />
Francis Codjoe, Eric Donkor<br />
<em>Medical Sciences</em> (2017-12-21) <a href="https://doi.org/c9zd">https://doi.org/c9zd</a><br />
DOI: <a href="https://doi.org/10.3390/medsci6010001">10.3390/medsci6010001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29267233">29267233</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5872158">PMC5872158</a></p>
</div>
<div id="ref-10jnMhzRm">
<p>6. <strong>Federal Action Plan on Antimicrobial Resistance</strong><br />
Public Health Agency of Canada<br />
(2015) <a href="https://www.canada.ca/en/health-canada/services/publications/drugs-health-products/federal-action-plan-antimicrobial-resistance-canada.html">https://www.canada.ca/en/health-canada/services/publications/drugs-health-products/federal-action-plan-antimicrobial-resistance-canada.html</a></p>
</div>
<div id="ref-3pJyRnMt">
<p>7. <strong>WHO Global Action Plan on Antimicrobial Resistance</strong><br />
World Health Organisation<br />
(2015) <a href="https://www.who.int/antimicrobial-resistance/global-action-plan/en/">https://www.who.int/antimicrobial-resistance/global-action-plan/en/</a></p>
</div>
<div id="ref-Pb5xYOu4">
<p>8. <strong>WHO Global Priority List of Antibiotic Resistant Bacteria</strong><br />
World Health Organisation<br />
(2017) <a href="https://www.who.int/medicines/publications/global-priority-list-antibiotic-resistant-bacteria/en/">https://www.who.int/medicines/publications/global-priority-list-antibiotic-resistant-bacteria/en/</a></p>
</div>
<div id="ref-bODq6dmJ">
<p>9. <strong>Antimicrobial Resistance in ESKAPE Pathogens</strong><br />
David M. P. De Oliveira, Brian M. Forde, Timothy J. Kidd, Patrick N. A. Harris, Mark A. Schembri, Scott A. Beatson, David L. Paterson, Mark J. Walker<br />
<em>Clinical Microbiology Reviews</em> (2020-05-13) <a href="https://doi.org/dzzh">https://doi.org/dzzh</a><br />
DOI: <a href="https://doi.org/10.1128/cmr.00181-19">10.1128/cmr.00181-19</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32404435">32404435</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7227449">PMC7227449</a></p>
</div>
<div id="ref-b2c0QSMI">
<p>10. <strong>Mechanisms of Antimicrobial Resistance in ESKAPE Pathogens</strong><br />
Sirijan Santajit, Nitaya Indrawattana<br />
<em>BioMed Research International</em> (2016) <a href="https://doi.org/f9h4q3">https://doi.org/f9h4q3</a><br />
DOI: <a href="https://doi.org/10.1155/2016/2475067">10.1155/2016/2475067</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27274985">27274985</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4871955">PMC4871955</a></p>
</div>
<div id="ref-aMfsvHsj">
<p>11. <strong>Federal Funding for the Study of Antimicrobial Resistance in Nosocomial Pathogens: No ESKAPE</strong><br />
Louis B. Rice<br />
<em>The Journal of Infectious Diseases</em> (2008-04-15) <a href="https://doi.org/frhgmx">https://doi.org/frhgmx</a><br />
DOI: <a href="https://doi.org/10.1086/533452">10.1086/533452</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18419525">18419525</a></p>
</div>
<div id="ref-oR3fXrzb">
<p>12. <strong>Extended-Spectrum β-Lactamases: a Clinical Update</strong><br />
David L. Paterson, Robert A. Bonomo<br />
<em>Clinical Microbiology Reviews</em> (2005-10) <a href="https://doi.org/d595m9">https://doi.org/d595m9</a><br />
DOI: <a href="https://doi.org/10.1128/cmr.18.4.657-686.2005">10.1128/cmr.18.4.657-686.2005</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16223952">16223952</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1265908">PMC1265908</a></p>
</div>
<div id="ref-hbKzfUrw">
<p>13. <strong>Canadian Nosocomial Infection Surveillance Program</strong><br />
Public Health Agency of Canada<br />
(2019) <a href="https://www.canada.ca/en/public-health/services/publications/science-research-data/summary-report-healthcare-associated-infection-antimicrobial-resistance-antimicrobial-use-surveillance-data-2013-2017.html">https://www.canada.ca/en/public-health/services/publications/science-research-data/summary-report-healthcare-associated-infection-antimicrobial-resistance-antimicrobial-use-surveillance-data-2013-2017.html</a></p>
</div>
<div id="ref-16ZzmUN7q">
<p>14. <strong>The structure of β-lactamases</strong><br />
Philosophical Transactions of the Royal Society of London. B, Biological Sciences<br />
(1980-05-16) <a href="https://doi.org/cdckks">https://doi.org/cdckks</a><br />
DOI: <a href="https://doi.org/10.1098/rstb.1980.0049">10.1098/rstb.1980.0049</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/6109327">6109327</a></p>
</div>
<div id="ref-2p1n2mr5">
<p>15. <strong>Updated Functional Classification of β-Lactamases</strong><br />
Karen Bush, George A. Jacoby<br />
<em>Antimicrobial Agents and Chemotherapy</em> (2009-12-07) <a href="https://doi.org/bp6dp2">https://doi.org/bp6dp2</a><br />
DOI: <a href="https://doi.org/10.1128/aac.01009-09">10.1128/aac.01009-09</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19995920">19995920</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825993">PMC2825993</a></p>
</div>
<div id="ref-UV1QqTS8">
<p>16. <strong>Class D OXA-48 Carbapenemase in Multidrug-Resistant Enterobacteria, Senegal</strong><br />
Olivier Moquet, Coralie Bouchiat, Alfred Kinana, Abdoulaye Seck, Omar Arouna, Raymond Bercion, Sebastien Breurec, Benoit Garin<br />
<em>Emerging Infectious Diseases</em> (2011-01) <a href="https://doi.org/cn235v">https://doi.org/cn235v</a><br />
DOI: <a href="https://doi.org/10.3201/eid1701.100244">10.3201/eid1701.100244</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21192883">21192883</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3204621">PMC3204621</a></p>
</div>
<div id="ref-FqGNAObZ">
<p>17. <strong>Rapid Identification of OXA-48 and OXA-163 Subfamilies in Carbapenem-Resistant Gram-Negative Bacilli with a Novel Immunochromatographic Lateral Flow Assay</strong><br />
Fernando Pasteran, Laurence Denorme, Isabelle Ote, Sonia Gomez, Denise De Belder, Youri Glupczynski, Pierre Bogaerts, Barbara Ghiglione, Pablo Power, Pascal Mertens, Alejandra Corso<br />
<em>Journal of Clinical Microbiology</em> (2016-08-17) <a href="https://doi.org/dbhs">https://doi.org/dbhs</a><br />
DOI: <a href="https://doi.org/10.1128/jcm.01175-16">10.1128/jcm.01175-16</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27535687">27535687</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5078564">PMC5078564</a></p>
</div>
<div id="ref-17k3PVuIy">
<p>18. <strong>Treatment Options for Carbapenem-Resistant Enterobacteriaceae Infections</strong><br />
Haley J. Morrill, Jason M. Pogue, Keith S. Kaye, Kerry L. LaPlante<br />
<em>Open Forum Infectious Diseases</em> (2015) <a href="https://doi.org/dbht">https://doi.org/dbht</a><br />
DOI: <a href="https://doi.org/10.1093/ofid/ofv050">10.1093/ofid/ofv050</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26125030">26125030</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4462593">PMC4462593</a></p>
</div>
<div id="ref-vQTzfitx">
<p>19. <strong>Towards Understanding MCR-like Colistin Resistance</strong><br />
Jian Sun, Huimin Zhang, Ya-Hong Liu, Youjun Feng<br />
<em>Trends in Microbiology</em> (2018-09) <a href="https://doi.org/gdqcfq">https://doi.org/gdqcfq</a><br />
DOI: <a href="https://doi.org/10.1016/j.tim.2018.02.006">10.1016/j.tim.2018.02.006</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29525421">29525421</a></p>
</div>
<div id="ref-11oxu50eO">
<p>20. <strong>mcr-1 identified in Avian Pathogenic Escherichia coli (APEC).</strong><br />
Nicolle Lima Barbieri, Daniel W Nielsen, Yvonne Wannemuehler, Tia Cavender, Ashraf Hussein, Shi-Gan Yan, Lisa K Nolan, Catherine M Logue<br />
<em>PloS one</em> (2017-03-06) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28264015">https://www.ncbi.nlm.nih.gov/pubmed/28264015</a><br />
DOI: <a href="https://doi.org/10.1371/journal.pone.0172997">10.1371/journal.pone.0172997</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28264015">28264015</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5338808">PMC5338808</a></p>
</div>
<div id="ref-nvbyXyPe">
<p>21. <strong>CARD 2020: antibiotic resistome surveillance with the comprehensive antibiotic resistance database</strong><br />
Brian P Alcock, Amogelang R Raphenya, Tammy TY Lau, Kara K Tsang, Mégane Bouchard, Arman Edalatmand, William Huynh, Anna-Lisa V Nguyen, Annie A Cheng, Sihan Liu, … Andrew G McArthur<br />
<em>Nucleic Acids Research</em> (2019-10-29) <a href="https://doi.org/ggckg6">https://doi.org/ggckg6</a><br />
DOI: <a href="https://doi.org/10.1093/nar/gkz935">10.1093/nar/gkz935</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31665441">31665441</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7145624">PMC7145624</a></p>
</div>
<div id="ref-x2VQlvOO">
<p>22. <strong>Proposal for assignment of allele numbers for mobile colistin resistance (mcr) genes</strong><br />
Sally R Partridge, Vincenzo Di Pilato, Yohei Doi, Michael Feldgarden, Daniel H Haft, William Klimke, Samir Kumar-Singh, Jian-Hua Liu, Surbhi Malhotra-Kumar, Arjun Prasad, … Basil Britto Xavier<br />
<em>Journal of Antimicrobial Chemotherapy</em> (2018-07-19) <a href="https://doi.org/dw7v">https://doi.org/dw7v</a><br />
DOI: <a href="https://doi.org/10.1093/jac/dky262">10.1093/jac/dky262</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30053115">30053115</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148208">PMC6148208</a></p>
</div>
<div id="ref-1AFSAuSwx">
<p>23. <strong>Molecular Characterization of OXA-20, a Novel Class D β-Lactamase, and Its Integron from Pseudomonas aeruginosa</strong><br />
Thierry Naas, Wladimir Sougakoff, Anne Casetta, Patrice Nordmann<br />
<em>Antimicrobial Agents and Chemotherapy</em> (1998-08-01) <a href="https://doi.org/dwsr">https://doi.org/dwsr</a><br />
DOI: <a href="https://doi.org/10.1128/aac.42.8.2074">10.1128/aac.42.8.2074</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/9687410">9687410</a></p>
</div>
<div id="ref-wapKbEHA">
<p>24. <strong>Author Correction: Recovery of nearly 8,000 metagenome-assembled genomes substantially expands the tree of life</strong><br />
Donovan H. Parks, Christian Rinke, Maria Chuvochina, Pierre-Alain Chaumeil, Ben J. Woodcroft, Paul N. Evans, Philip Hugenholtz, Gene W. Tyson<br />
<em>Nature Microbiology</em> (2017-12-12) <a href="https://doi.org/c8rq">https://doi.org/c8rq</a><br />
DOI: <a href="https://doi.org/10.1038/s41564-017-0083-5">10.1038/s41564-017-0083-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29234139">29234139</a></p>
</div>
<div id="ref-DTtDOvle">
<p>25. <strong>MetaBAT, an efficient tool for accurately reconstructing single genomes from complex microbial communities</strong><br />
Dongwan D. Kang, Jeff Froula, Rob Egan, Zhong Wang<br />
<em>PeerJ</em> (2015-08-27) <a href="https://doi.org/gdf329">https://doi.org/gdf329</a><br />
DOI: <a href="https://doi.org/10.7717/peerj.1165">10.7717/peerj.1165</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26336640">26336640</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4556158">PMC4556158</a></p>
</div>
<div id="ref-DfIRBmdF">
<p>26. <strong>Recovery of genomes from metagenomes via a dereplication, aggregation and scoring strategy</strong><br />
Christian M. K. Sieber, Alexander J. Probst, Allison Sharrar, Brian C. Thomas, Matthias Hess, Susannah G. Tringe, Jillian F. Banfield<br />
<em>Nature Microbiology</em> (2018-05-28) <a href="https://doi.org/gfwwfg">https://doi.org/gfwwfg</a><br />
DOI: <a href="https://doi.org/10.1038/s41564-018-0171-1">10.1038/s41564-018-0171-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29807988">29807988</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6786971">PMC6786971</a></p>
</div>
<div id="ref-wrBRBdFb">
<p>27. <strong>Recovery of nearly 8,000 metagenome-assembled genomes substantially expands the tree of life</strong><br />
Donovan H. Parks, Christian Rinke, Maria Chuvochina, Pierre-Alain Chaumeil, Ben J. Woodcroft, Paul N. Evans, Philip Hugenholtz, Gene W. Tyson<br />
<em>Nature Microbiology</em> (2017-09-11) <a href="https://doi.org/cczd">https://doi.org/cczd</a><br />
DOI: <a href="https://doi.org/10.1038/s41564-017-0012-7">10.1038/s41564-017-0012-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28894102">28894102</a></p>
</div>
<div id="ref-s9ycaHcq">
<p>28. <strong>Basic local alignment search tool</strong><br />
Stephen F. Altschul, Warren Gish, Webb Miller, Eugene W. Myers, David J. Lipman<br />
<em>Journal of Molecular Biology</em> (1990-10) <a href="https://doi.org/cnsjsz">https://doi.org/cnsjsz</a><br />
DOI: <a href="https://doi.org/10.1016/s0022-2836(05)80360-2">10.1016/s0022-2836(05)80360-2</a></p>
</div>
<div id="ref-1DMuBNrog">
<p>29. <strong>MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform</strong><br />
K. Katoh<br />
<em>Nucleic Acids Research</em> (2002-07-15) <a href="https://doi.org/dbpmq3">https://doi.org/dbpmq3</a><br />
DOI: <a href="https://doi.org/10.1093/nar/gkf436">10.1093/nar/gkf436</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12136088">12136088</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC135756">PMC135756</a></p>
</div>
<div id="ref-GCsU1nyf">
<p>30. <strong>trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses</strong><br />
S. Capella-Gutierrez, J. M. Silla-Martinez, T. Gabaldon<br />
<em>Bioinformatics</em> (2009-06-08) <a href="https://doi.org/bjhdh7">https://doi.org/bjhdh7</a><br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btp348">10.1093/bioinformatics/btp348</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19505945">19505945</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2712344">PMC2712344</a></p>
</div>
<div id="ref-JOWSuu8G">
<p>31. <strong>IQ-TREE: A Fast and Effective Stochastic Algorithm for Estimating Maximum-Likelihood Phylogenies</strong><br />
Lam-Tung Nguyen, Heiko A. Schmidt, Arndt von Haeseler, Bui Quang Minh<br />
<em>Molecular Biology and Evolution</em> (2014-11-03) <a href="https://doi.org/f3srtd">https://doi.org/f3srtd</a><br />
DOI: <a href="https://doi.org/10.1093/molbev/msu300">10.1093/molbev/msu300</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25371430">25371430</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4271533">PMC4271533</a></p>
</div>
<div id="ref-5F7ii9Ji">
<p>32. <strong>ETE 3: Reconstruction, Analysis, and Visualization of Phylogenomic Data</strong><br />
Jaime Huerta-Cepas, François Serra, Peer Bork<br />
<em>Molecular Biology and Evolution</em> (2016-02-26) <a href="https://doi.org/gfzpph">https://doi.org/gfzpph</a><br />
DOI: <a href="https://doi.org/10.1093/molbev/msw046">10.1093/molbev/msw046</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26921390">26921390</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4868116">PMC4868116</a></p>
</div>
<div id="ref-JT9eRkR8">
<p>33. <strong>Substrate Recognition by a Colistin Resistance Enzyme from Moraxella catarrhalis</strong><br />
Peter J. Stogios, Georgina Cox, Haley L. Zubyk, Elena Evdokimova, Zdzislaw Wawrzak, Gerard D. Wright, Alexei Savchenko<br />
<em>ACS Chemical Biology</em> (2018-04-09) <a href="https://doi.org/gdnrmn">https://doi.org/gdnrmn</a><br />
DOI: <a href="https://doi.org/10.1021/acschembio.8b00116">10.1021/acschembio.8b00116</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29631403">29631403</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6197822">PMC6197822</a></p>
</div>
<div id="ref-16EwPCiJT">
<p>34. <strong>The Genus Aeromonas: Taxonomy, Pathogenicity, and Infection</strong><br />
J. Michael Janda, Sharon L. Abbott<br />
<em>Clinical Microbiology Reviews</em> (2010-01) <a href="https://doi.org/cbnfww">https://doi.org/cbnfww</a><br />
DOI: <a href="https://doi.org/10.1128/cmr.00039-09">10.1128/cmr.00039-09</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20065325">20065325</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2806660">PMC2806660</a></p>
</div>
<div id="ref-8uz5m0fP">
<p>35. <strong>Natural antimicrobial susceptibility patterns and biochemical profiles of Leclercia adecarboxylata strains</strong><br />
I Stock, S Burak, B Wiedemann<br />
<em>Clinical Microbiology and Infection</em> (2004-08) <a href="https://doi.org/b2zr9k">https://doi.org/b2zr9k</a><br />
DOI: <a href="https://doi.org/10.1111/j.1469-0691.2004.00892.x">10.1111/j.1469-0691.2004.00892.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15301675">15301675</a></p>
</div>
<div id="ref-BjoH1Vii">
<p>36. <strong>The Genus Psychrobacter</strong><br />
Elliot Juni<br />
<em>The Prokaryotes</em> (1992) <a href="https://doi.org/dbpr">https://doi.org/dbpr</a><br />
DOI: <a href="https://doi.org/10.1007/978-1-4757-2191-1_12">10.1007/978-1-4757-2191-1_12</a></p>
</div>
<div id="ref-WLzELTTa">
<p>37. <strong>Novel Psychrobacter Species from Antarctic Ornithogenic Soils</strong><br />
J. P. BOWMAN, J. CAVANAGH, J. J. AUSTIN, K. SANDERSON<br />
<em>International Journal of Systematic Bacteriology</em> (1996-10-01) <a href="https://doi.org/fd9rb9">https://doi.org/fd9rb9</a><br />
DOI: <a href="https://doi.org/10.1099/00207713-46-4-841">10.1099/00207713-46-4-841</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8863407">8863407</a></p>
</div>
<div id="ref-2NZAmp2H">
<p>38. <strong>Psychrobacter vallis sp. nov. and Psychrobacter aquaticus sp. nov., from Antarctica</strong><br />
Sisinthy Shivaji, Gundlapalli S. N. Reddy, Korpole Suresh, Pratima Gupta, Suresh Chintalapati, Peter Schumann, Erko Stackebrandt, Genki I. Matsumoto<br />
<em>International Journal of Systematic and Evolutionary Microbiology</em> (2005-03-01) <a href="https://doi.org/cdq9w4">https://doi.org/cdq9w4</a><br />
DOI: <a href="https://doi.org/10.1099/ijs.0.03030-0">10.1099/ijs.0.03030-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15774658">15774658</a></p>
</div>
<div id="ref-Ovca71PG">
<p>39. <strong>Stenotrophomonas maltophilia: an Emerging Global Opportunistic Pathogen</strong><br />
J. S. Brooke<br />
<em>Clinical Microbiology Reviews</em> (2012-01-01) <a href="https://doi.org/fxx2tt">https://doi.org/fxx2tt</a><br />
DOI: <a href="https://doi.org/10.1128/cmr.00019-11">10.1128/cmr.00019-11</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22232370">22232370</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3255966">PMC3255966</a></p>
</div>
<div id="ref-KEhNzdys">
<p>40. <strong>Dissemination and Characteristics of a Novel Plasmid-Encoded Carbapenem-Hydrolyzing Class D β-Lactamase, OXA-436, Found in Isolates from Four Patients at Six Different Hospitals in Denmark</strong><br />
Ørjan Samuelsen, Frank Hansen, Bettina Aasnæs, Henrik Hasman, Bjarte Aarmo Lund, Hanna-Kirsti S. Leiros, Berit Lilje, Jessin Janice, Lotte Jakobsen, Pia Littauer, … Anette M. Hammerum<br />
<em>Antimicrobial Agents and Chemotherapy</em> (2017-10-23) <a href="https://doi.org/db7x">https://doi.org/db7x</a><br />
DOI: <a href="https://doi.org/10.1128/aac.01260-17">10.1128/aac.01260-17</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29061750">29061750</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5740310">PMC5740310</a></p>
</div>
<div id="ref-12zFifp5x">
<p>41. <strong>On the (im)possibility of reconstructing plasmids from whole-genome short-read sequencing data</strong><br />
Sergio Arredondo-Alonso, Rob J. Willems, Willem van Schaik, Anita C. Schürch<br />
<em>Microbial Genomics</em> (2017-10-01) <a href="https://doi.org/gf6b63">https://doi.org/gf6b63</a><br />
DOI: <a href="https://doi.org/10.1099/mgen.0.000128">10.1099/mgen.0.000128</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29177087">29177087</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5695206">PMC5695206</a></p>
</div>
<div id="ref-eGJmNg9p">
<p>42. <strong>Resistance gene naming and numbering: is it a new gene or not?</strong><br />
Ruth M. Hall, Stefan Schwarz<br />
<em>Journal of Antimicrobial Chemotherapy</em> (2015-10-28) <a href="https://doi.org/f8gn4r">https://doi.org/f8gn4r</a><br />
DOI: <a href="https://doi.org/10.1093/jac/dkv351">10.1093/jac/dkv351</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26510717">26510717</a></p>
</div>
<div id="ref-EhIOtkvv">
<p>43. <strong>Comment on: Resistance gene naming and numbering: is it a new gene or not?</strong><br />
Benjamin A. Evans<br />
<em>Journal of Antimicrobial Chemotherapy</em> (2016-03-06) <a href="https://doi.org/dw7r">https://doi.org/dw7r</a><br />
DOI: <a href="https://doi.org/10.1093/jac/dkw037">10.1093/jac/dkw037</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26953334">26953334</a></p>
</div>
</div>
<!-- default theme -->

<style>
    /* import google fonts */
    @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
    @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

    /* -------------------------------------------------- */
    /* global */
    /* -------------------------------------------------- */

    /* all elements */
    * {
        /* force sans-serif font unless specified otherwise */
        font-family: "Open Sans", "Helvetica", sans-serif;

        /* prevent text inflation on some mobile browsers */
        -webkit-text-size-adjust: none !important;
        -moz-text-size-adjust: none !important;
        -o-text-size-adjust: none !important;
        text-size-adjust: none !important;
    }

    @media only screen {
        /* "page" element */
        body {
            position: relative;
            box-sizing: border-box;
            font-size: 12pt;
            line-height: 1.5;
            max-width: 8.5in;
            margin: 20px auto;
            padding: 40px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* "page" element */
        body {
            padding: 20px;
            margin: 0;
            border-radius: 0;
            border: none;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
            background: none;
        }
    }

    /* -------------------------------------------------- */
    /* headings */
    /* -------------------------------------------------- */

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
        margin: 20px 0;
        padding: 0;
        font-weight: bold;
    }

    /* biggest heading */
    h1 {
        margin: 40px 0;
        text-align: center;
    }

    /* second biggest heading */
    h2 {
        margin-top: 30px;
        padding-bottom: 5px;
        border-bottom: solid 1px #bdbdbd;
    }

    /* -------------------------------------------------- */
    /* manuscript header */
    /* -------------------------------------------------- */

    /* manuscript title */
    header > h1 {
        margin: 0;
    }

    /* manuscript title caption text (ie "automatically generated on") */
    header + p {
        text-align: center;
        margin-top: 10px;
    }

    /* -------------------------------------------------- */
    /* text elements */
    /* -------------------------------------------------- */

    /* links */
    a {
        color: #2196f3;
    }

    /* normal links (not empty, not button link, not syntax highlighting link) */
    a:not(:empty):not(.button):not(.sourceLine) {
        padding-left: 1px;
        padding-right: 1px;
    }

    /* superscripts and subscripts */
    sub,
    sup {
        /* prevent from affecting line height */
        line-height: 0;
    }

    /* unordered and ordered lists*/
    ul,
    ol {
        padding-left: 20px;
    }

    /* class for styling text semibold */
    .semibold {
        font-weight: 600;
    }

    /* class for styling elements horizontally left aligned */
    .left {
        display: block;
        text-align: left;
        margin-left: auto;
        margin-right: 0;
        justify-content: left;
    }

    /* class for styling elements horizontally centered */
    .center {
        display: block;
        text-align: center;
        margin-left: auto;
        margin-right: auto;
        justify-content: center;
    }

    /* class for styling elements horizontally right aligned */
    .right {
        display: block;
        text-align: right;
        margin-left: 0;
        margin-right: auto;
        justify-content: right;
    }

    /* -------------------------------------------------- */
    /* section elements */
    /* -------------------------------------------------- */

    /* horizontal divider line */
    hr {
        border: none;
        height: 1px;
        background: #bdbdbd;
    }

    /* paragraphs, horizontal dividers, figures, tables, code */
    p,
    hr,
    figure,
    table,
    pre {
        /* treat all as "paragraphs", with consistent vertical margins */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* figures */
    /* -------------------------------------------------- */

    /* figure */
    figure {
        max-width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure caption */
    figcaption {
        padding: 0;
        padding-top: 10px;
    }

    /* figure image element */
    figure img {
        max-width: 100%;
        display: block;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure auto-number */
    img + figcaption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* tables */
    /* -------------------------------------------------- */

    /* table */
    table {
        border-collapse: collapse;
        border-spacing: 0;
        width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* table cells */
    th,
    td {
        border: solid 1px #bdbdbd;
        padding: 10px;
        /* squash table if too wide for page by forcing line breaks */
        word-break: break-all;
        word-break: break-word;
    }

    /* header row and even rows */
    th,
    tr:nth-child(2n) {
        background-color: #fafafa;
    }

    /* odd rows */
    tr:nth-child(2n + 1) {
        background-color: #ffffff;
    }

    /* table caption */
    caption {
        text-align: left;
        padding: 0;
        padding-bottom: 10px;
    }

    /* table auto-number */
    table > caption > span:first-of-type,
    div.table_wrapper > table > caption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* code */
    /* -------------------------------------------------- */

    /* multi-line code block */
    pre {
        padding: 10px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
        break-inside: avoid;
        text-align: left;
    }

    /* inline code, ie code within normal text */
    :not(pre) > code {
        padding: 0 4px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
    }

    /* code text */
    /* apply all children, to reach syntax highlighting sub-elements */
    code,
    code * {
        /* force monospace font */
        font-family: "Source Code Pro", "Courier New", monospace;
    }

    /* -------------------------------------------------- */
    /* quotes */
    /* -------------------------------------------------- */

    /* quoted text */
    blockquote {
        margin: 0;
        padding: 0;
        border-left: 4px solid #bdbdbd;
        padding-left: 16px;
        break-inside: avoid;
    }

    /* -------------------------------------------------- */
    /* banners */
    /* -------------------------------------------------- */

    /* info banners */
    .banner {
        box-sizing: border-box;
        display: block;
        position: relative;
        width: 100%;
        margin-top: 20px;
        margin-bottom: 20px;
        padding: 20px;
        text-align: center;
    }

    /* paragraph in banner */
    .banner > p {
        margin: 0;
    }

    /* -------------------------------------------------- */
    /* highlight colors */
    /* -------------------------------------------------- */

    .white {
        background: #ffffff;
    }
    .lightgrey {
        background: #eeeeee;
    }
    .grey {
        background: #757575;
    }
    .darkgrey {
        background: #424242;
    }
    .black {
        background: #000000;
    }
    .lightred {
        background: #ffcdd2;
    }
    .lightyellow {
        background: #ffecb3;
    }
    .lightgreen {
        background: #dcedc8;
    }
    .lightblue {
        background: #e3f2fd;
    }
    .lightpurple {
        background: #f3e5f5;
    }
    .red {
        background: #f44336;
    }
    .orange {
        background: #ff9800;
    }
    .yellow {
        background: #ffeb3b;
    }
    .green {
        background: #4caf50;
    }
    .blue {
        background: #2196f3;
    }
    .purple {
        background: #9c27b0;
    }
    .white,
    .lightgrey,
    .lightred,
    .lightyellow,
    .lightgreen,
    .lightblue,
    .lightpurple,
    .orange,
    .yellow,
    .white a,
    .lightgrey a,
    .lightred a,
    .lightyellow a,
    .lightgreen a,
    .lightblue a,
    .lightpurple a,
    .orange a,
    .yellow a {
        color: #000000;
    }
    .grey,
    .darkgrey,
    .black,
    .red,
    .green,
    .blue,
    .purple,
    .grey a,
    .darkgrey a,
    .black a,
    .red a,
    .green a,
    .blue a,
    .purple a {
        color: #ffffff;
    }

    /* -------------------------------------------------- */
    /* buttons */
    /* -------------------------------------------------- */

    /* class for styling links like buttons */
    .button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        margin: 5px;
        padding: 10px 20px;
        font-size: 0.75em;
        font-weight: 600;
        text-transform: uppercase;
        text-decoration: none;
        letter-spacing: 1px;
        background: none;
        color: #2196f3;
        border: solid 1px #bdbdbd;
        border-radius: 5px;
    }

    /* buttons when hovered */
    .button:hover:not([disabled]),
    .icon_button:hover:not([disabled]) {
        cursor: pointer;
        background: #f5f5f5;
    }

    /* buttons when disabled */
    .button[disabled],
    .icon_button[disabled] {
        opacity: 0.35;
        pointer-events: none;
    }

    /* class for styling buttons containg only single icon */
    .icon_button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        text-decoration: none;
        margin: 0;
        padding: 0;
        background: none;
        border-radius: 5px;
        border: none;
        width: 20px;
        height: 20px;
        min-width: 20px;
        min-height: 20px;
    }

    /* icon button inner svg image */
    .icon_button > svg {
        height: 16px;
    }

    /* -------------------------------------------------- */
    /* icons */
    /* -------------------------------------------------- */

    /* class for styling icons inline with text */
    .inline_icon {
        height: 1em;
        position: relative;
        top: 0.125em;
    }

    /* -------------------------------------------------- */
    /* print control */
    /* -------------------------------------------------- */

    @media print {
        @page {
            /* suggested printing margin */
            margin: 0.5in;
        }

        /* document and "page" elements */
        html, body {
            margin: 0;
            padding: 0;
            width: 100%;
            height: 100%;
        }

        /* "page" element */
        body {
            font-size: 11pt !important;
            line-height: 1.35;
        }

        /* all headings */
        h1,
        h2,
        h3,
        h4,
        h5,
        h6 {
            margin: 15px 0;
        }

        /* heading 1 */
        h1 {
            font-size: 1.75em;
        }

        /* heading 2 */
        h2 {
            font-size: 1.25em;
            margin-top: 0;
        }

        /* heading 3 */
        h3 {
            font-size: 1.10em;
        }

        /* figures and tables */
        figure, table {
            font-size: 0.85em;
        }

        /* table cells */
        th,
        td {
            padding: 5px;
        }

        /* shrink font awesome icons */
        i.fas,
        i.fab,
        i.far,
        i.fal {
            transform: scale(0.85);
        }

        /* decrease banner margins */
        .banner {
            margin-top: 15px;
            margin-bottom: 15px;
            padding: 15px;
        }

        /* class for centering an element vertically on its own page */
        .page_center {
            margin: auto;
            width: 100%;
            height: 100%;
            display: flex;
            align-items: center;
            vertical-align: middle;
            break-before: page;
            break-after: page;
        }

        /* always insert a page break before the element */
        .page_break_before {
            break-before: page;
        }

        /* always insert a page break after the element */
        .page_break_after {
            break-after: page;
        }

        /* avoid page break before the element */
        .page_break_before_avoid {
            break-before: avoid;
        }

        /* avoid page break after the element */
        .page_break_after_avoid {
            break-after: avoid;
        }

        /* avoid page break inside the element */
        .page_break_inside_avoid {
            break-inside: avoid;
        }
    }

    /* -------------------------------------------------- */
    /* override pandoc css quirks */
    /* -------------------------------------------------- */

    .sourceCode {
        /* prevent unsightly overflow in wide code blocks */
        overflow: auto !important;
    }

    div.sourceCode {
        /* prevent background fill on top-most code block  container */
        background: none !important;
    }

    .sourceCode * {
        /* force consistent line spacing */
        line-height: 1.5 !important;
    }

    div.sourceCode {
        /* style code block margins same as <pre> element */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* mathjax */
    /* -------------------------------------------------- */

    /* mathjax containers */
    .math.display > span:not(.MathJax_Preview) {
        /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
        display: flex !important;
        overflow-x: auto !important;
        overflow-y: hidden !important;
        justify-content: center;
        align-items: center;
        margin: 0 !important;
    }

    /* right click menu */
    .MathJax_Menu {
        border-radius: 5px !important;
        border: solid 1px #bdbdbd !important;
        box-shadow: none !important;
    }

    /* equation auto-number */
    span[id^="eq:"] > span.math.display + span {
        font-weight: 600;
    }

    /* equation */
    span[id^="eq:"] > span.math.display > span {
        /* nudge to make room for equation auto-number */
        margin-right: 40px !important;
    }

    /* -------------------------------------------------- */
    /* table scroll plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* table wrapper */
        .table_wrapper {
            /* show scrollbar on tables if necessary to prevent overflow */
            overflow: auto;
            width: 100%;
            margin: 20px 0;
        }

        /* table within table wrapper */
        .table_wrapper table,
        .table_wrapper table * {
            /* don't break table words */
            word-break: normal !important;
        }

        .table_wrapper > table {
            /* move margins from table to table_wrapper to allow margin collapsing */
            margin: 0;
        }
    }

    /* -------------------------------------------------- */
    /* anchors plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anchor button */
        .anchor {
            opacity: 0;
            margin-left: 5px;
        }

        /* anchor buttons within <h2>'s */
        h2 .anchor {
            margin-left: 10px;
        }

        /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
        *:hover > .anchor,
        .anchor:hover,
        .anchor:focus {
            opacity: 1;
        }

        /* anchor button when hovered */
        .anchor:hover {
            cursor: pointer;
        }
    }

    /* always show anchor button on devices with no mouse/hover ability */
    @media (hover: none) {
        .anchor {
            opacity: 1;
        }
    }

    /* always hide anchor button on print */
    @media only print {
        .anchor {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* accordion plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* accordion arrow button */
        .accordion_arrow {
            margin-right: 10px;
        }

        /* arrow icon when <h2> data-collapsed attribute true */
        h2[data-collapsed="true"] > .accordion_arrow > svg {
            transform: rotate(-90deg);
        }

        /* all elements (except <h2>'s) when data-collapsed attribute true */
        *:not(h2)[data-collapsed="true"] {
            display: none;
        }

        /* accordion arrow button when hovered and <h2>'s when hovered */
        .accordion_arrow:hover,
        h2[data-collapsed="true"]:hover,
        h2[data-collapsed="false"]:hover {
            cursor: pointer;
        }
    }

    /* always hide accordion arrow button on print */
    @media only print {
        .accordion_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* tooltips plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* tooltip container */
        #tooltip {
            position: absolute;
            width: 50%;
            min-width: 240px;
            max-width: 75%;
            z-index: 1;
        }

        /* tooltip content */
        #tooltip_content {
            margin-bottom: 5px;
            padding: 20px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
            word-break: break-word;
        }

        /* tooltip copy of paragraphs and figures */
        #tooltip_content > p,
        #tooltip_content > figure {
            margin: 0;
            max-height: 320px;
            overflow-y: auto;
        }

        /* tooltip copy of <img> */
        #tooltip_content > figure > img {
            max-height: 260px;
        }

        /* navigation bar */
        #tooltip_nav_bar {
            margin-top: 10px;
            text-align: center;
        }

        /* navigation bar previous/next buton */
        #tooltip_nav_bar > .icon_button {
            position: relative;
            top: 3px;
        }

        /* navigation bar previous button */
        #tooltip_nav_bar > .icon_button:first-of-type {
            margin-right: 5px;
        }

        /* navigation bar next button */
        #tooltip_nav_bar > .icon_button:last-of-type {
            margin-left: 5px;
        }
    }

    /* always hide tooltip on print */
    @media only print {
        #tooltip {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* jump to first plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* jump button */
        .jump_arrow {
            position: relative;
            top: 0.125em;
            margin-right: 5px;
        }
    }

    /* always hide jump button on print */
    @media only print {
        .jump_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* link highlight plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anything with data-highlighted attribute true */
        [data-highlighted="true"] {
            background: #ffeb3b;
        }

        /* anything with data-selected attribute true */
        [data-selected="true"] {
            background: #ff8a65 !important;
        }

        /* animation definition for glow */
        @keyframes highlight_glow {
            0% {
                background: none;
            }
            10% {
                background: #bbdefb;
            }
            100% {
                background: none;
            }
        }

        /* anything with data-glow attribute true */
        [data-glow="true"] {
            animation: highlight_glow 2s;
        }
    }

    /* -------------------------------------------------- */
    /* table of contents plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* toc panel */
        #toc_panel {
            box-sizing: border-box;
            position: fixed;
            top: 0;
            left: 0;
            background: #ffffff;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            z-index: 2;
        }

        /* toc panel when closed */
        #toc_panel[data-open="false"] {
            min-width: 60px;
            width: 60px;
            height: 60px;
            border-right: solid 1px #bdbdbd;
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc panel when open */
        #toc_panel[data-open="true"] {
            min-width: 260px;
            max-width: 480px;
            /* keep panel edge consistent distance away from "page" edge */
            width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
            bottom: 0;
            border-right: solid 1px #bdbdbd;
        }

        /* toc panel header */
        #toc_header {
            box-sizing: border-box;
            display: flex;
            flex-direction: row;
            align-items: center;
            height: 60px;
            margin: 0;
            padding: 20px;
        }

        /* toc panel header when hovered */
        #toc_header:hover {
            cursor: pointer;
        }

        /* toc panel header when panel open */
        #toc_panel[data-open="true"] > #toc_header {
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc open/close header button */
        #toc_button {
            margin-right: 20px;
        }

        /* hide toc list and header text when closed */
        #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
        #toc_panel[data-open="false"] > #toc_list {
            display: none;
        }

        /* toc list of entries */
        #toc_list {
            box-sizing: border-box;
            width: 100%;
            padding: 20px;
            position: absolute;
            top: calc(60px + 1px);
            bottom: 0;
            overflow: auto;
        }

        /* toc entry, link to section in document */
        .toc_link {
            display: block;
            padding: 5px;
            position: relative;
            font-weight: 600;
            text-decoration: none;
        }

        /* toc entry when hovered or when "viewed" */
        .toc_link:hover,
        .toc_link[data-viewing="true"] {
            background: #f5f5f5;
        }

        /* toc entry, level 1 indentation */
        .toc_link[data-level="1"] {
            margin-left: 0;
        }

        /* toc entry, level 2 indentation */
        .toc_link[data-level="2"] {
            margin-left: 20px;
        }

        /* toc entry, level 3 indentation */
        .toc_link[data-level="3"] {
            margin-left: 40px;
        }

        /* toc entry, level 4 indentation */
        .toc_link[data-level="4"] {
            margin-left: 60px;
        }

        /* toc entry bullets */
        #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
            position: absolute;
            left: -15px;
            top: -1px;
            font-size: 1.5em;
        }

        /* toc entry, level 2 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
            content: "\2022";
        }

        /* toc entry, level 3 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
            content: "\25AB";
        }

        /* toc entry, level 4 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
            content: "-";
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* push <body> ("page") element down to make room for toc icon */
        .toc_body_nudge {
            padding-top: 60px;
        }

        /* toc icon when panel closed and not hovered */
        #toc_panel[data-open="false"]:not(:hover) {
            background: rgba(255, 255, 255, 0.75);
        }
    }

    /* always hide toc panel on print */
    @media only print {
        #toc_panel {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* lightbox plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* regular <img> in document when hovered */
        .lightbox_document_img:hover {
            cursor: pointer;
        }

        .body_no_scroll {
            overflow: hidden !important;
        }

        /* screen overlay */
        #lightbox_overlay {
            display: flex;
            flex-direction: column;
            position: fixed;
            left: 0;
            top: 0;
            right: 0;
            bottom: 0;
            background: rgba(0, 0, 0, 0.75);
            z-index: 3;
        }

        /* middle area containing lightbox image */
        #lightbox_image_container {
            flex-grow: 1;
            display: flex;
            justify-content: center;
            align-items: center;
            overflow: hidden;
            position: relative;
            padding: 20px;
        }

        /* bottom area containing caption */
        #lightbox_bottom_container {
            display: flex;
            justify-content: center;
            align-items: center;
            height: 100px;
            min-height: 100px;
            max-height: 100px;
            background: rgba(0, 0, 0, 0.5);
        }

        /* image number info text box */
        #lightbox_number_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            left: 2px;
            top: 0;
            z-index: 4;
        }

        /* zoom info text box */
        #lightbox_zoom_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            right: 2px;
            top: 0;
            z-index: 4;
        }

        /* copy of image caption */
        #lightbox_caption {
            box-sizing: border-box;
            display: inline-block;
            width: 100%;
            max-height: 100%;
            padding: 10px 0;
            text-align: center;
            overflow-y: auto;
            color: #ffffff;
        }

        /* navigation previous/next button */
        .lightbox_button {
            width: 100px;
            height: 100%;
            min-width: 100px;
            min-height: 100%;
            color: #ffffff;
        }

        /* navigation previous/next button when hovered */
        .lightbox_button:hover {
            background: none !important;
        }

        /* navigation button icon */
        .lightbox_button > svg {
            height: 25px;
        }

        /* figure auto-number */
        #lightbox_caption > span:first-of-type {
            font-weight: bold;
            margin-right: 5px;
        }

        /* lightbox image when hovered */
        #lightbox_img:hover {
            cursor: grab;
        }

        /* lightbox image when grabbed */
        #lightbox_img:active {
            cursor: grabbing;
        }
    }

    /* when on screen < 480px wide */
    @media only screen and (max-width: 480px) {
        /* make navigation buttons skinnier on small screens to make more room for caption text */
        .lightbox_button {
            width: 50px;
            min-width: 50px;
        }
    }

    /* always hide lightbox on print */
    @media only print {
        #lightbox_overlay {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* hypothesis (annotations) plugin */
    /* -------------------------------------------------- */

    /* side panel */
    .annotator-frame {
        width: 280px !important;
        z-index: 0 !important;
    }

    /* match highlight color to rest of theme */
    .annotator-highlights-always-on .annotator-hl {
        background-color: #ffeb3b !important;
    }

    /* match focused color to rest of theme */
    .annotator-hl.annotator-hl-focused {
        background-color: #ff8a65 !important;
    }

    /* match bucket bar color to rest of theme */
    .annotator-bucket-bar {
        background: #f5f5f5 !important;
    }

    /* always hide toolbar and tooltip on print */
    @media only print {
        .annotator-frame {
            display: none !important;
        }

        hypothesis-adder {
            display: none !important;
        }
    }
</style>
<!-- table scroll plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin allows tables that are too wide to fit within
        // the page to have a scrollbar instead of being squashed to fit.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tableScroll';

        // default plugin options
        const options = {
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // wrap each table in a container
            const tables = document.querySelectorAll('table');
            for (const table of tables)
                wrapElement(table).classList.add('table_wrapper');
            // table_wrapper CSS class in theme file provides scroll
            // functionality
        }

        // wrap element in div and return div
        function wrapElement(element) {
            const parent = element.parentNode;
            const wrapper = document.createElement('div');
            parent.replaceChild(wrapper, element);
            wrapper.appendChild(element);
            return wrapper;
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>
<!-- anchors plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin adds an anchor next to each of a certain type
        // of element that provides a human-readable url to that specific
        // item/position in the document (eg "manuscript.html#abstract"). It
        // also makes it such that scrolling out of view of a target removes
        // its identifier from the url.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'anchors';

        // default plugin options
        const options = {
            // which types of elements to add anchors next to, in
            // "document.querySelector" format
            typesQuery: 'h1, h2, h3, figure, table',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // add anchor to each element of specified types
            const elements = document.querySelectorAll(options.typesQuery);
            for (const element of elements)
                addAnchor(element);

            // attach scroll listener to window
            window.addEventListener('scroll', onScroll);
        }

        // when window is scrolled
        function onScroll() {
            // if url has hash and user has scrolled out of view of hash
            // target, remove hash from url
            const target = getHashTarget();
            if (target) {
                if (
                    target.getBoundingClientRect().top > window.innerHeight ||
                    target.getBoundingClientRect().bottom < 0
                )
                    history.pushState(null, null, ' ');
            }
        }

        // add anchor to element
        function addAnchor(element) {
            let withId; // element with unique id
            let addTo; // element to add anchor button to

            // if figure or table, modify withId and addTo to get expected
            // elements
            if (element.tagName.toLowerCase() === 'figure') {
                withId = element.querySelector('img');
                addTo = element.querySelector('figcaption');
            } else if (element.tagName.toLowerCase() === 'table') {
                withId =
                    element.previousElementSibling ||
                    element.parentNode.previousElementSibling;
                addTo = element.querySelector('caption');
            }

            withId = withId || element;
            addTo = addTo || element;
            const id = withId.id || withId.name || null;

            // do not add anchor if element doesn't have assigned id.
            // id is generated by pandoc and is assumed to be unique and
            // human-readable
            if (!id)
                return;

            // create anchor button
            const anchor = document.createElement('a');
            anchor.innerHTML = document.querySelector('.icon_link').innerHTML;
            anchor.title = 'Link to this part of the document';
            anchor.classList.add('icon_button', 'anchor');
            anchor.dataset.ignore = 'true';
            anchor.href = '#' + id;
            addTo.appendChild(anchor);
        }

        // get element that is target of link or url hash
        function getHashTarget() {
            const hash = window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector(
                '[id="' + id + '"], [name="' + id + '"]'
            );
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (hash.indexOf('#fig:') === 0)
                target = target.parentNode;
            if (hash.indexOf('#tbl:') === 0)
                target = target.nextElementSibling;

            return target;
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- link icon -->

<template class="icon_link">
    <!-- modified from: https://fontawesome.com/icons/link -->
    <svg width="16" height="16" viewBox="0 0 512 512">
        <path
            fill="currentColor"
            d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
        ></path>
    </svg>
</template>
<!-- accordion plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin allows sections of content under <h2> headings
        // to be collapsible.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'accordion';

        // default plugin options
        const options = {
            // whether to always start expanded ('false'), always start
            // collapsed ('true'), or start collapsed when screen small ('auto')
            startCollapsed: 'auto',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // run through each <h2> heading
            const headings = document.querySelectorAll('h2');
            for (const heading of headings) {
                addArrow(heading);

                // start expanded/collapsed based on option
                if (
                    options.startCollapsed === 'true' ||
                    (options.startCollapsed === 'auto' && isSmallScreen())
                )
                    collapseHeading(heading);
                else
                    expandHeading(heading);
            }

            // attach hash change listener to window
            window.addEventListener('hashchange', onHashChange);
        }

        // when hash (eg manuscript.html#introduction) changes
        function onHashChange() {
            const target = getHashTarget();
            if (target)
                goToElement(target);
        }

        // add arrow to heading
        function addArrow(heading) {
            // add arrow button
            const arrow = document.createElement('button');
            arrow.innerHTML = document.querySelector(
                '.icon_angle_down'
            ).innerHTML;
            arrow.classList.add('icon_button', 'accordion_arrow');
            heading.insertBefore(arrow, heading.firstChild);

            // attach click listener to heading and button
            heading.addEventListener('click', onHeadingClick);
            arrow.addEventListener('click', onArrowClick);
        }

        // determine if on mobile-like device with small screen
        function isSmallScreen() {
            return Math.min(window.innerWidth, window.innerHeight) < 480;
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // get element that is target of hash
        function getHashTarget(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector(
                '[id="' + id + '"], [name="' + id + '"]'
            );
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (hash.indexOf('#fig:') === 0)
                target = target.parentNode;
            if (hash.indexOf('#tbl:') === 0)
                target = target.nextElementSibling;

            return target;
        }

        // when <h2> heading is clicked
        function onHeadingClick(event) {
            // only collapse if <h2> itself is target of click (eg, user did
            // not click on anchor within <h2>)
            if (event.target === this)
                toggleCollapse(this);
        }

        // when arrow button is clicked
        function onArrowClick() {
            toggleCollapse(this.parentNode);
        }

        // collapse section if expanded, expand if collapsed
        function toggleCollapse(heading) {
            if (heading.dataset.collapsed === 'false')
                collapseHeading(heading);
            else
                expandHeading(heading);
        }

        // elements to exclude from collapse, such as table of contents panel,
        // hypothesis panel, etc
        const exclude = '#toc_panel, div.annotator-frame, #lightbox_overlay';

        // collapse section
        function collapseHeading(heading) {
            heading.setAttribute('data-collapsed', 'true');
            const children = getChildren(heading);
            for (const child of children)
                child.setAttribute('data-collapsed', 'true');
        }

        // expand section
        function expandHeading(heading) {
            heading.setAttribute('data-collapsed', 'false');
            const children = getChildren(heading);
            for (const child of children)
                child.setAttribute('data-collapsed', 'false');
        }

        // get list of elements between this <h2> and next <h2> or <h1>
        // ("children" of the <h2> section)
        function getChildren(heading) {
            return nextUntil(heading, 'h2, h1', exclude);
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get list of elements after a start element up to element matching
        // query
        function nextUntil(element, query, exclude) {
            const elements = [];
            while (element = element.nextElementSibling, element) {
                if (element.matches(query))
                    break;
                if (!element.matches(exclude))
                    elements.push(element);
            }
            return elements;
        }

        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
    <!-- modified from: https://fontawesome.com/icons/angle-down -->
    <svg width="16" height="16" viewBox="0 0 448 512">
        <path
            fill="currentColor"
            d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
        ></path>
    </svg>
</template>
<!-- tooltips plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when the user hovers or
        // focuses a link to a citation or figure, a tooltip appears with a
        // preview of the reference content, along with arrows to navigate
        // between instances of the same reference in the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tooltips';

        // default plugin options
        const options = {
            // whether user must click off to close tooltip instead of just
            // un-hovering
            clickClose: 'false',
            // delay (in ms) between opening and closing tooltip
            delay: '100',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            const links = getLinks();
            for (const link of links) {
                // attach hover and focus listeners to link
                link.addEventListener('mouseover', onLinkHover);
                link.addEventListener('mouseleave', onLinkUnhover);
                link.addEventListener('focus', onLinkFocus);
                link.addEventListener('touchend', onLinkTouch);
            }

            // attach mouse, key, and resize listeners to window
            window.addEventListener('mousedown', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('keyup', onKeyUp);
            window.addEventListener('resize', onResize);
        }

        // when link is hovered
        function onLinkHover() {
            // function to open tooltip
            const delayOpenTooltip = function() {
                openTooltip(this);
            }.bind(this);

            // run open function after delay
            this.openTooltipTimer = window.setTimeout(
                delayOpenTooltip,
                options.delay
            );
        }

        // when mouse leaves link
        function onLinkUnhover() {
            // cancel opening tooltip
            window.clearTimeout(this.openTooltipTimer);

            // don't close on unhover if option specifies
            if (options.clickClose === 'true')
                return;

            // function to close tooltip
            const delayCloseTooltip = function() {
                // if tooltip open and if mouse isn't over tooltip, close
                const tooltip = document.getElementById('tooltip');
                if (tooltip && !tooltip.matches(':hover'))
                    closeTooltip();
            };

            // run close function after delay
            this.closeTooltipTimer = window.setTimeout(
                delayCloseTooltip,
                options.delay
            );
        }

        // when link is focused (tabbed to)
        function onLinkFocus(event) {
            openTooltip(this);
        }

        // when link is touched on touch screen
        function onLinkTouch(event) {
            // attempt to force hover state on first tap always, and trigger
            // regular link click (and navigation) on second tap
            if (event.target === document.activeElement)
                event.target.click();
            else {
                document.activeElement.blur();
                event.target.focus();
            }
            if (event.cancelable)
                event.preventDefault();
            event.stopPropagation();
            return false;
        }

        // when mouse is clicked anywhere in window
        function onClick(event) {
            closeTooltip();
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            switch (event.key) {
                // trigger click of prev button
                case 'ArrowLeft':
                    const prevButton = document.getElementById(
                        'tooltip_prev_button'
                    );
                    if (prevButton)
                        prevButton.click();
                    break;
                // trigger click of next button
                case 'ArrowRight':
                    const nextButton = document.getElementById(
                        'tooltip_next_button'
                    );
                    if (nextButton)
                        nextButton.click();
                    break;
                // close on esc
                case 'Escape':
                    closeTooltip();
                    break;
            }
        }

        // when window is resized or zoomed
        function onResize() {
            closeTooltip();
        }

        // get all links of types we wish to handle
        function getLinks() {
            const queries = [];
            // exclude buttons, anchor links, toc links, etc
            const exclude =
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            queries.push('a[href^="#ref-"]' + exclude); // citation links
            queries.push('a[href^="#fig:"]' + exclude); // figure links
            const query = queries.join(', ');
            return document.querySelectorAll(query);
        }

        // get links with same target, get index of link in set, get total
        // same links
        function getSameLinks(link) {
            const sameLinks = [];
            const links = getLinks();
            for (const otherLink of links) {
                if (
                    otherLink.getAttribute('href') === link.getAttribute('href')
                )
                    sameLinks.push(otherLink);
            }

            return {
                elements: sameLinks,
                index: sameLinks.indexOf(link),
                total: sameLinks.length
            };
        }

        // open tooltip
        function openTooltip(link) {
            // delete tooltip if it exists, start fresh
            closeTooltip();

            // make tooltip element
            const tooltip = makeTooltip(link);

            // if source couldn't be found and tooltip not made, exit
            if (!tooltip)
                return;

            // make navbar elements
            const navBar = makeNavBar(link);
            if (navBar)
                tooltip.firstElementChild.appendChild(navBar);

            // attach tooltip to page
            document.body.appendChild(tooltip);

            // position tooltip
            const position = function() {
                positionTooltip(link);
            };
            position();

            // if tooltip contains images, position again after they've loaded
            const imgs = tooltip.querySelectorAll('img');
            for (const img of imgs)
                img.addEventListener('load', position);
        }

        // close (delete) tooltip
        function closeTooltip() {
            const tooltip = document.getElementById('tooltip');
            if (tooltip)
                tooltip.remove();
        }

        // make tooltip
        function makeTooltip(link) {
            // get target element that link points to
            const source = getSource(link);

            // if source can't be found, exit
            if (!source)
                return;

            // create new tooltip
            const tooltip = document.createElement('div');
            tooltip.id = 'tooltip';
            const tooltipContent = document.createElement('div');
            tooltipContent.id = 'tooltip_content';
            tooltip.appendChild(tooltipContent);

            // make copy of source node and put in tooltip
            const sourceCopy = makeCopy(source);
            tooltipContent.appendChild(sourceCopy);

            // attach mouse event listeners
            tooltip.addEventListener('click', onTooltipClick);
            tooltip.addEventListener('mousedown', onTooltipClick);
            tooltip.addEventListener('touchstart', onTooltipClick);
            tooltip.addEventListener('mouseleave', onTooltipUnhover);

            // (for interaction with lightbox plugin)
            // transfer click on tooltip copied img to original img
            const sourceImg = source.querySelector('img');
            const sourceCopyImg = sourceCopy.querySelector('img');
            if (sourceImg && sourceCopyImg) {
                const clickImg = function() {
                    sourceImg.click();
                    closeTooltip();
                };
                sourceCopyImg.addEventListener('click', clickImg);
            }

            return tooltip;
        }

        // make carbon copy of html dom element
        function makeCopy(source) {
            const sourceCopy = source.cloneNode(true);

            // delete elements marked with ignore (eg anchor and jump buttons)
            const deleteFromCopy = sourceCopy.querySelectorAll(
                '[data-ignore="true"]'
            );
            for (const element of deleteFromCopy)
                element.remove();

            // delete certain element attributes
            const attributes = [
                'id',
                'data-collapsed',
                'data-selected',
                'data-highlighted',
                'data-glow'
            ];
            for (const attribute of attributes) {
                sourceCopy.removeAttribute(attribute);
                const elements = sourceCopy.querySelectorAll(
                    '[' + attribute + ']'
                );
                for (const element of elements)
                    element.removeAttribute(attribute);
            }

            return sourceCopy;
        }

        // when tooltip is clicked
        function onTooltipClick(event) {
            // when user clicks on tooltip, stop click from transferring
            // outside of tooltip (eg, click off to close tooltip, or eg click
            // off to unhighlight same refs)
            event.stopPropagation();
        }

        // when tooltip is unhovered
        function onTooltipUnhover(event) {
            if (options.clickClose === 'true')
                return;

            // make sure new mouse/touch/focus no longer over tooltip or any
            // element within it
            const tooltip = document.getElementById('tooltip');
            if (!tooltip)
                return;
            if (this.contains(event.relatedTarget))
                return;

            closeTooltip();
        }

        // make nav bar to go betwen prev/next instances of same reference
        function makeNavBar(link) {
            // find other links to the same source
            const sameLinks = getSameLinks(link);

            // don't show nav bar when singular reference
            if (sameLinks.total <= 1)
                return;

            // find prev/next links with same target
            const prevLink = getPrevLink(link, sameLinks);
            const nextLink = getNextLink(link, sameLinks);

            // create nav bar
            const navBar = document.createElement('div');
            navBar.id = 'tooltip_nav_bar';
            const text = sameLinks.index + 1 + ' of ' + sameLinks.total;

            // create nav bar prev/next buttons
            const prevButton = document.createElement('button');
            const nextButton = document.createElement('button');
            prevButton.id = 'tooltip_prev_button';
            nextButton.id = 'tooltip_next_button';
            prevButton.title =
                'Jump to the previous occurence of this item in the document [←]';
            nextButton.title =
                'Jump to the next occurence of this item in the document [→]';
            prevButton.classList.add('icon_button');
            nextButton.classList.add('icon_button');
            prevButton.innerHTML = document.querySelector(
                '.icon_caret_left'
            ).innerHTML;
            nextButton.innerHTML = document.querySelector(
                '.icon_caret_right'
            ).innerHTML;
            navBar.appendChild(prevButton);
            navBar.appendChild(document.createTextNode(text));
            navBar.appendChild(nextButton);

            // attach click listeners to buttons
            prevButton.addEventListener('click', function() {
                onPrevNextClick(link, prevLink);
            });
            nextButton.addEventListener('click', function() {
                onPrevNextClick(link, nextLink);
            });

            return navBar;
        }

        // get previous link with same target
        function getPrevLink(link, sameLinks) {
            if (!sameLinks)
                sameLinks = getSameLinks(link);
            // wrap index to other side if < 1
            let index;
            if (sameLinks.index - 1 >= 0)
                index = sameLinks.index - 1;
            else
                index = sameLinks.total - 1;
            return sameLinks.elements[index];
        }

        // get next link with same target
        function getNextLink(link, sameLinks) {
            if (!sameLinks)
                sameLinks = getSameLinks(link);
            // wrap index to other side if > total
            let index;
            if (sameLinks.index + 1 <= sameLinks.total - 1)
                index = sameLinks.index + 1;
            else
                index = 0;
            return sameLinks.elements[index];
        }

        // get element that is target of link or url hash
        function getSource(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector(
                '[id="' + id + '"], [name="' + id + '"]'
            );
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (hash.indexOf('#ref-') === 0)
                target = target.querySelector('p');
            else if (hash.indexOf('#fig:') === 0)
                target = target.parentNode;
            else if (hash.indexOf('#tbl:') === 0)
                return;

            return target;
        }

        // when prev/next arrow button is clicked
        function onPrevNextClick(link, prevNextLink) {
            if (link && prevNextLink)
                goToElement(prevNextLink, window.innerHeight * 0.5);
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // determine position to place tooltip based on link position in
        // viewport and tooltip size
        function positionTooltip(link, left, top) {
            const tooltipElement = document.getElementById('tooltip');
            if (!tooltipElement)
                return;

            // get convenient vars for position/dimensions of
            // link/tooltip/page/view
            link = getRectInPage(link);
            const tooltip = getRectInPage(tooltipElement);
            const view = getRectInPage();

            // horizontal positioning
            if (left)
                // use explicit value
                left = left;
            else if (link.left + tooltip.width < view.right)
                // fit tooltip to right of link
                left = link.left;
            else if (link.right - tooltip.width > view.left)
                // fit tooltip to left of link
                left = link.right - tooltip.width;
            // center tooltip in view
            else
                left = (view.right - view.left) / 2 - tooltip.width / 2;

            // vertical positioning
            if (top)
                // use explicit value
                top = top;
            else if (link.top - tooltip.height > view.top)
                // fit tooltip above link
                top = link.top - tooltip.height;
            else if (link.bottom + tooltip.height < view.bottom)
                // fit tooltip below link
                top = link.bottom;
            else {
                // center tooltip in view
                top = view.top + view.height / 2 - tooltip.height / 2;
                // nudge off of link to left/right if possible
                if (link.right + tooltip.width < view.right)
                    left = link.right;
                else if (link.left - tooltip.width > view.left)
                    left = link.left - tooltip.width;
            }

            tooltipElement.style.left = left + 'px';
            tooltipElement.style.top = top + 'px';
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get position of element relative to page
        function getRectInPage(element) {
            const rect = getRectInView(element);
            const body = getRectInView(document.body);

            const newRect = {};
            newRect.left = rect.left - body.left;
            newRect.top = rect.top - body.top;
            newRect.right = rect.right - body.left;
            newRect.bottom = rect.bottom - body.top;
            newRect.width = rect.width;
            newRect.height = rect.height;

            return newRect;
        }

        // (for interaction with accordion plugin)
        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // (for interaction with accordion plugin)
        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // (for interaction with accordion plugin)
        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
    <!-- modified from: https://fontawesome.com/icons/caret-left -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
        ></path>
    </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
    <!-- modified from: https://fontawesome.com/icons/caret-right -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
        ></path>
    </svg>
</template>
<!-- jump to first plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin adds a button next to each reference entry,
        // figure, and table that jumps the page to the first occurrence of a
        // link to that item in the manuscript.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'jumpToFirst';

        // default plugin options
        const options = {
            // whether to add buttons next to reference entries
            references: 'true',
            // whether to add buttons next to figures
            figures: 'true',
            // whether to add buttons next to tables
            tables: 'true',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            if (options.references !== 'false')
                makeReferenceButtons();
            if (options.figures !== 'false')
                makeFigureButtons();
            if (options.tables !== 'false')
                makeTableButtons();
        }

        // when jump button clicked
        function onButtonClick() {
            const first = getFirstOccurrence(this.dataset.id);
            if (!first)
                return;

            // update url hash so navigating "back" in history will return
            // user to jump button
            window.location.hash = this.dataset.id;
            // scroll to link
            window.setTimeout(function() {
                goToElement(first, window.innerHeight * 0.5);
            }, 0);
        }

        // get first occurence of link to item in document
        function getFirstOccurrence(id) {
            let query = 'a';
            query += '[href="#' + id + '"]';
            // exclude buttons, anchor links, toc links, etc
            query +=
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            return document.querySelector(query);
        }

        // add button next to each reference entry
        function makeReferenceButtons() {
            const references = document.querySelectorAll('div[id^="ref-"]');
            for (const reference of references) {
                // get reference id and element to add button to
                const id = reference.id;
                const container = reference.firstElementChild;
                const first = getFirstOccurrence(id);

                // if can't find link to reference, ignore
                if (!first)
                    continue;

                // make jump button
                let button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this reference in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.innerHTML = button.outerHTML + container.innerHTML;
                button = container.firstElementChild;
                button.addEventListener('click', onButtonClick);
            }
        }

        // add button next to each figure
        function makeFigureButtons() {
            const figures = document.querySelectorAll('img[id^="fig:"]');
            for (const figure of figures) {
                // get figure id and element to add button to
                const id = figure.id;
                const container = figure.nextElementSibling;
                const first = getFirstOccurrence(id);

                // if can't find link to figure, ignore
                if (!first)
                    continue;

                // make jump button
                const button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this figure in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.insertBefore(button, container.firstElementChild);
                button.addEventListener('click', onButtonClick);
            }
        }

        // add button next to each figure
        function makeTableButtons() {
            const tables = document.querySelectorAll('a[name^="tbl:"]');
            for (const table of tables) {
                // get ref id and element to add button to
                const id = table.name;
                const container = table.nextElementSibling.querySelector(
                    'caption'
                );
                const first = getFirstOccurrence(id);

                // if can't find link to table, ignore
                if (!first)
                    continue;

                // make jump button
                const button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this table in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.insertBefore(button, container.firstElementChild);
                button.addEventListener('click', onButtonClick);
            }
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // (for interaction with accordion plugin)
        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
    <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
    <svg width="16" height="16" viewBox="0 0 320 512">
        <path
            fill="currentColor"
            d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
        ></path>
    </svg>
</template>
<!-- link highlight plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when a user hovers or
        // focuses a link, other links that have the same target will be
        // highlighted. It also makes it such that when clicking a link, the
        // target of the link (eg reference, figure, table) is briefly
        // highlighted.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'linkHighlight';

        // default plugin options
        const options = {
            // whether to also highlight links that go to external urls
            externalLinks: 'false',
            // whether user must click off to unhighlight instead of just
            // un-hovering
            clickUnhighlight: 'false',
            // whether to also highlight links that are unique
            highlightUnique: 'true',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            const links = getLinks();
            for (const link of links) {
                // attach mouse and focus listeners to link
                link.addEventListener('mouseenter', onLinkFocus);
                link.addEventListener('focus', onLinkFocus);
                link.addEventListener('mouseleave', onLinkUnhover);
            }

            // attach click and hash change listeners to window
            window.addEventListener('click', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('hashchange', onHashChange);

            // run hash change on window load in case user has navigated
            // directly to hash
            onHashChange();
        }

        // when link is focused (tabbed to) or hovered
        function onLinkFocus() {
            highlight(this);
        }

        // when link is unhovered
        function onLinkUnhover() {
            if (options.clickUnhighlight !== 'true')
                unhighlightAll();
        }

        // when the mouse is clicked anywhere in window
        function onClick(event) {
            unhighlightAll();
        }

        // when hash (eg manuscript.html#introduction) changes
        function onHashChange() {
            const target = getHashTarget();
            if (target)
                glowElement(target);
        }

        // get element that is target of link or url hash
        function getHashTarget(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector(
                '[id="' + id + '"], [name="' + id + '"]'
            );
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (hash.indexOf('#fig:') === 0)
                target = target.parentNode;
            else if (hash.indexOf('#tbl:') === 0)
                target = target.nextElementSibling.querySelector('caption');

            return target;
        }

        // start glow sequence on an element
        function glowElement(element) {
            const startGlow = function() {
                onGlowEnd();
                element.dataset.glow = 'true';
                element.addEventListener('animationend', onGlowEnd);
            };
            const onGlowEnd = function() {
                element.removeAttribute('data-glow');
                element.removeEventListener('animationend', onGlowEnd);
            };
            startGlow();
        }

        // highlight link and all others with same target
        function highlight(link) {
            // force unhighlight all to start fresh
            unhighlightAll();

            // get links with same target
            if (!link)
                return;
            const sameLinks = getSameLinks(link);

            // if link unique and option is off, exit and don't highlight
            if (sameLinks.length <= 1 && options.highlightUnique !== 'true')
                return;

            // highlight all same links, and "select" (special highlight) this
            // one
            for (const sameLink of sameLinks) {
                if (sameLink === link)
                    sameLink.setAttribute('data-selected', 'true');
                else
                    sameLink.setAttribute('data-highlighted', 'true');
            }
        }

        // unhighlight all links
        function unhighlightAll() {
            const links = getLinks();
            for (const link of links) {
                link.setAttribute('data-selected', 'false');
                link.setAttribute('data-highlighted', 'false');
            }
        }

        // get links with same target
        function getSameLinks(link) {
            const results = [];
            const links = getLinks();
            for (const otherLink of links) {
                if (
                    otherLink.getAttribute('href') === link.getAttribute('href')
                )
                    results.push(otherLink);
            }
            return results;
        }

        // get all links of types we wish to handle
        function getLinks() {
            let query = 'a';
            if (options.externalLinks !== 'true')
                query += '[href^="#"]';
            // exclude buttons, anchor links, toc links, etc
            query +=
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            return document.querySelectorAll(query);
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>
<!-- table of contents plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin provides a "table of contents" (toc) panel on
        // the side of the document that allows the user to conveniently
        // navigate between sections of the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tableOfContents';

        // default plugin options
        const options = {
            // which types of elements to add links for, in
            // "document.querySelector" format
            typesQuery: 'h1, h2, h3',
            // whether default behavior is to be closed ('false'), open
            // ('true'), or only open when screen wide enough to fit panel
            // ('auto'). note: still always starts closed when page loads.
            open: 'auto',
            // if list item is more than this many characters, text will be
            // truncated
            charLimit: '50',
            // whether or not to show bullets next to each toc item
            bullets: 'false',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // make toc panel and populate with entries (links to document
            // sections)
            const panel = makePanel();
            if (!panel)
                return;
            makeEntries(panel);
            document.body.insertBefore(panel, document.body.firstChild);

            closePanel();

            // attach click, scroll, and hash change listeners to window
            window.addEventListener('click', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('scroll', onScroll);
            window.addEventListener('hashchange', onScroll);
            window.addEventListener('keyup', onKeyUp);
            onScroll();

            // add class to push document body down out of way of toc button
            document.body.classList.add('toc_body_nudge');
        }

        // determine if screen wide enough to fit toc panel
        function isSmallScreen() {
            // in default theme:
            // 816px = 8.5in = width of "page" (<body>) element
            // 260px = min width of toc panel (*2 for both sides of <body>)
            return window.innerWidth < 816 + 260 * 2;
        }

        // open/close panel based on option and screen size
        function openOrClosePanel() {
            if (
                options.open === 'true' ||
                (options.open === 'auto' && !isSmallScreen())
            )
                openPanel();
            else
                closePanel();
        }

        // when mouse is clicked anywhere in window
        function onClick() {
            const panel = document.getElementById('toc_panel');
            if (!panel)
                return;

            if (panel.dataset.open === 'true')
                openOrClosePanel();
        }

        // when window is scrolled or hash changed
        function onScroll() {
            highlightViewed();
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            // close on esc
            if (event.key === 'Escape')
                closePanel();
        }

        // find entry of currently viewed document section in toc and highlight
        function highlightViewed() {
            const firstId = getFirstInView(options.typesQuery);

            // get toc entries (links), unhighlight all, then highlight viewed
            const list = document.getElementById('toc_list');
            if (!firstId || !list)
                return;
            const links = list.querySelectorAll('a');
            for (const link of links)
                link.dataset.viewing = 'false';
            const link = list.querySelector('a[href="#' + firstId + '"]');
            if (!link)
                return;
            link.dataset.viewing = 'true';
        }

        // get first or previous toc listed element in top half of view
        function getFirstInView(query) {
            // get all elements matching query and with id
            const elements = document.querySelectorAll(query);
            const elementsWithIds = [];
            for (const element of elements) {
                if (element.id)
                    elementsWithIds.push(element);
            }


            // get first or previous element in top half of view
            for (let i = 0; i < elementsWithIds.length; i++) {
                const element = elementsWithIds[i];
                const prevElement = elementsWithIds[Math.max(0, i - 1)];
                if (element.getBoundingClientRect().top >= 0) {
                    if (
                        element.getBoundingClientRect().top <
                        window.innerHeight / 2
                    )
                        return element.id;
                    else
                        return prevElement.id;
                }
            }
        }

        // make panel
        function makePanel() {
            // create panel
            const panel = document.createElement('div');
            panel.id = 'toc_panel';
            if (options.bullets === 'true')
                panel.dataset.bullets = 'true';

            // create header
            const header = document.createElement('div');
            header.id = 'toc_header';

            // create toc button
            const button = document.createElement('button');
            button.id = 'toc_button';
            button.innerHTML = document.querySelector(
                '.icon_th_list'
            ).innerHTML;
            button.classList.add('icon_button');

            // create header text
            const text = document.createElement('h3');
            text.innerHTML = 'Table of Contents';

            // create container for toc list
            const list = document.createElement('div');
            list.id = 'toc_list';

            // attach click listeners
            panel.addEventListener('click', onPanelClick);
            header.addEventListener('click', onHeaderClick);
            button.addEventListener('click', onButtonClick);

            // attach elements
            header.appendChild(button);
            header.appendChild(text);
            panel.appendChild(header);
            panel.appendChild(list);

            return panel;
        }

        // create toc entries (links) to each element of the specified types
        function makeEntries(panel) {
            const elements = document.querySelectorAll(options.typesQuery);
            for (const element of elements) {
                // do not add link if element doesn't have assigned id
                if (!element.id)
                    continue;

                // create link/list item
                const link = document.createElement('a');
                link.classList.add('toc_link');
                switch (element.tagName.toLowerCase()) {
                    case 'h1':
                        link.dataset.level = '1';
                        break;
                    case 'h2':
                        link.dataset.level = '2';
                        break;
                    case 'h3':
                        link.dataset.level = '3';
                        break;
                    case 'h4':
                        link.dataset.level = '4';
                        break;
                }
                link.title = element.innerText;
                let text = element.innerText;
                if (text.length > options.charLimit)
                    text = text.slice(0, options.charLimit) + '...';
                link.innerHTML = text;
                link.href = '#' + element.id;
                link.addEventListener('click', onLinkClick);

                // attach link
                panel.querySelector('#toc_list').appendChild(link);
            }
        }

        // when panel is clicked
        function onPanelClick(event) {
            // stop click from propagating to window/document and closing panel
            event.stopPropagation();
        }

        // when header itself is clicked
        function onHeaderClick(event) {
            togglePanel();
        }

        // when button is clicked
        function onButtonClick(event) {
            togglePanel();
            // stop header underneath button from also being clicked
            event.stopPropagation();
        }

        // when link is clicked
        function onLinkClick() {
            openOrClosePanel();
        }

        // open panel if closed, close if opened
        function togglePanel() {
            const panel = document.getElementById('toc_panel');
            if (!panel)
                return;

            if (panel.dataset.open === 'true')
                closePanel();
            else
                openPanel();
        }

        // open panel
        function openPanel() {
            const panel = document.getElementById('toc_panel');
            if (panel)
                panel.dataset.open = 'true';
        }

        // close panel
        function closePanel() {
            const panel = document.getElementById('toc_panel');
            if (panel)
                panel.dataset.open = 'false';
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- th list icon -->

<template class="icon_th_list">
    <!-- modified from: https://fontawesome.com/icons/th-list -->
    <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
        <path
            fill="currentColor"
            d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
            tabindex="-1"
        ></path>
    </svg>
</template>
<!-- lightbox plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when a user clicks on an
        // image, the image fills the screen and the user can pan/drag/zoom
        // the image and navigate between other images in the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'lightbox';

        // default plugin options
        const options = {
            // list of possible zoom/scale factors
            zoomSteps:
                '0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1,' +
                '1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8',
            // whether to fit image to view ('fit'), display at 100% and shrink
            // if necessary ('shrink'), or always display at 100% ('100')
            defaultZoom: 'fit',
            // whether to zoom in/out toward center of view ('true') or mouse
            // ('false')
            centerZoom: 'false',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // run through each <img> element
            const imgs = document.querySelectorAll('figure > img');
            let count = 1;
            for (const img of imgs) {
                img.classList.add('lightbox_document_img');
                img.dataset.number = count;
                img.dataset.total = imgs.length;
                img.addEventListener('click', openLightbox);
                count++;
            }

            // attach mouse and key listeners to window
            window.addEventListener('mousemove', onWindowMouseMove);
            window.addEventListener('keyup', onKeyUp);
        }

        // when mouse is moved anywhere in window
        function onWindowMouseMove(event) {
            window.mouseX = event.clientX;
            window.mouseY = event.clientY;
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            switch (event.key) {
                // trigger click of prev button
                case 'ArrowLeft':
                    const prevButton = document.getElementById(
                        'lightbox_prev_button'
                    );
                    if (prevButton)
                        prevButton.click();
                    break;
                // trigger click of next button
                case 'ArrowRight':
                    const nextButton = document.getElementById(
                        'lightbox_next_button'
                    );
                    if (nextButton)
                        nextButton.click();
                    break;
                // close on esc
                case 'Escape':
                    closeLightbox();
                    break;
            }
        }

        // open lightbox
        function openLightbox() {
            const lightbox = makeLightbox(this);
            if (!lightbox)
                return;

            blurBody(lightbox);
            document.body.appendChild(lightbox);
        }

        // make lightbox
        function makeLightbox(img) {
            // delete lightbox if it exists, start fresh
            closeLightbox();

            // create screen overlay containing lightbox
            const overlay = document.createElement('div');
            overlay.id = 'lightbox_overlay';

            // create image info boxes
            const numberInfo = document.createElement('div');
            const zoomInfo = document.createElement('div');
            numberInfo.id = 'lightbox_number_info';
            zoomInfo.id = 'lightbox_zoom_info';

            // create container for image
            const imageContainer = document.createElement('div');
            imageContainer.id = 'lightbox_image_container';
            const lightboxImg = makeLightboxImg(
                img,
                imageContainer,
                numberInfo,
                zoomInfo
            );
            imageContainer.appendChild(lightboxImg);

            // create bottom container for caption and navigation buttons
            const bottomContainer = document.createElement('div');
            bottomContainer.id = 'lightbox_bottom_container';
            const caption = makeCaption(img);
            const prevButton = makePrevButton(img);
            const nextButton = makeNextButton(img);
            bottomContainer.appendChild(prevButton);
            bottomContainer.appendChild(caption);
            bottomContainer.appendChild(nextButton);

            // attach top middle and bottom to overlay
            overlay.appendChild(numberInfo);
            overlay.appendChild(zoomInfo);
            overlay.appendChild(imageContainer);
            overlay.appendChild(bottomContainer);

            return overlay;
        }

        // make <img> object that is intuitively draggable and zoomable
        function makeLightboxImg(
            sourceImg,
            container,
            numberInfoBox,
            zoomInfoBox
        ) {
            // create copy of source <img>
            const img = sourceImg.cloneNode(true);
            img.classList.remove('lightbox_document_img');
            img.removeAttribute('id');
            img.removeAttribute('width');
            img.removeAttribute('height');
            img.style.position = 'unset';
            img.style.margin = '0';
            img.style.padding = '0';
            img.style.width = '';
            img.style.height = '';
            img.style.minWidth = '';
            img.style.minHeight = '';
            img.style.maxWidth = '';
            img.style.maxHeight = '';
            img.id = 'lightbox_img';

            // build sorted list of unique zoomSteps, always including a 100%
            let zoomSteps = [];
            const optionsZooms = options.zoomSteps.split(/[^0-9.]/);
            for (const optionZoom of optionsZooms) {
                const newZoom = parseFloat(optionZoom);
                if (newZoom && !zoomSteps.includes(newZoom))
                    zoomSteps.push(newZoom);
            }
            if (!zoomSteps.includes(1))
                zoomSteps.push(1);
            zoomSteps = zoomSteps.sort(function sortNumber(a, b) {
                return a - b;
            });

            // <img> object property variables
            let zoom = 1;
            let translateX = 0;
            let translateY = 0;
            let clickMouseX = undefined;
            let clickMouseY = undefined;
            let clickTranslateX = undefined;
            let clickTranslateY = undefined;

            updateNumberInfo();

            // update image numbers displayed in info box
            function updateNumberInfo() {
                numberInfoBox.innerHTML =
                    sourceImg.dataset.number + ' of ' + sourceImg.dataset.total;
            }

            // update zoom displayed in info box
            function updateZoomInfo() {
                let zoomInfo = zoom * 100;
                if (!Number.isInteger(zoomInfo))
                    zoomInfo = zoomInfo.toFixed(2);
                zoomInfoBox.innerHTML = zoomInfo + '%';
            }

            // move to closest zoom step above current zoom
            const zoomIn = function() {
                for (const zoomStep of zoomSteps) {
                    if (zoomStep > zoom) {
                        zoom = zoomStep;
                        break;
                    }
                }
                updateTransform();
            };

            // move to closest zoom step above current zoom
            const zoomOut = function() {
                zoomSteps.reverse();
                for (const zoomStep of zoomSteps) {
                    if (zoomStep < zoom) {
                        zoom = zoomStep;
                        break;
                    }
                }
                zoomSteps.reverse();

                updateTransform();
            };

            // update display of <img> based on scale/translate properties
            const updateTransform = function() {
                // set transform
                img.style.transform =
                    'translate(' +
                    (translateX || 0) +
                    'px,' +
                    (translateY || 0) +
                    'px) scale(' +
                    (zoom || 1) +
                    ')';

                // get new width/height after scale
                const rect = img.getBoundingClientRect();
                // limit translate
                translateX = Math.max(translateX, -rect.width / 2);
                translateX = Math.min(translateX, rect.width / 2);
                translateY = Math.max(translateY, -rect.height / 2);
                translateY = Math.min(translateY, rect.height / 2);

                // set transform
                img.style.transform =
                    'translate(' +
                    (translateX || 0) +
                    'px,' +
                    (translateY || 0) +
                    'px) scale(' +
                    (zoom || 1) +
                    ')';

                updateZoomInfo();
            };

            // fit <img> to container
            const fit = function() {
                // no x/y offset, 100% zoom by default
                translateX = 0;
                translateY = 0;
                zoom = 1;

                // widths of <img> and container
                const imgWidth = img.naturalWidth;
                const imgHeight = img.naturalHeight;
                const containerWidth = parseFloat(
                    window.getComputedStyle(container).width
                );
                const containerHeight = parseFloat(
                    window.getComputedStyle(container).height
                );

                // how much zooming is needed to fit <img> to container
                const xRatio = imgWidth / containerWidth;
                const yRatio = imgHeight / containerHeight;
                const maxRatio = Math.max(xRatio, yRatio);
                const newZoom = 1 / maxRatio;

                // fit <img> to container according to option
                if (options.defaultZoom === 'shrink') {
                    if (maxRatio > 1)
                        zoom = newZoom;
                } else if (options.defaultZoom === 'fit')
                    zoom = newZoom;

                updateTransform();
            };

            // when mouse wheel is rolled anywhere in container
            const onContainerWheel = function(event) {
                if (!event)
                    return;

                // let ctrl + mouse wheel to zoom behave as normal
                if (event.ctrlKey)
                    return;

                // prevent normal scroll behavior
                event.preventDefault();
                event.stopPropagation();

                // point around which to scale img
                const viewRect = container.getBoundingClientRect();
                const viewX = (viewRect.left + viewRect.right) / 2;
                const viewY = (viewRect.top + viewRect.bottom) / 2;
                const originX = options.centerZoom === 'true' ? viewX : mouseX;
                const originY = options.centerZoom === 'true' ? viewY : mouseY;

                // get point on image under origin
                const oldRect = img.getBoundingClientRect();
                const oldPercentX = (originX - oldRect.left) / oldRect.width;
                const oldPercentY = (originY - oldRect.top) / oldRect.height;

                // increment/decrement zoom
                if (event.deltaY < 0)
                    zoomIn();
                if (event.deltaY > 0)
                    zoomOut();

                // get offset between previous image point and origin
                const newRect = img.getBoundingClientRect();
                const offsetX =
                    originX - (newRect.left + newRect.width * oldPercentX);
                const offsetY =
                    originY - (newRect.top + newRect.height * oldPercentY);

                // translate image to keep image point under origin
                translateX += offsetX;
                translateY += offsetY;

                // perform translate
                updateTransform();
            };

            // when container is clicked
            function onContainerClick(event) {
                // if container itself is target of click, and not other
                // element above it
                if (event.target === this)
                    closeLightbox();
            }

            // when mouse button is pressed on image
            const onImageMouseDown = function(event) {
                // store original mouse position relative to image
                clickMouseX = window.mouseX;
                clickMouseY = window.mouseY;
                clickTranslateX = translateX;
                clickTranslateY = translateY;
                event.stopPropagation();
                event.preventDefault();
            };

            // when mouse button is released anywhere in window
            const onWindowMouseUp = function(event) {
                // reset original mouse position
                clickMouseX = undefined;
                clickMouseY = undefined;
                clickTranslateX = undefined;
                clickTranslateY = undefined;

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('mouseup', onWindowMouseUp);
            };

            // when mouse is moved anywhere in window
            const onWindowMouseMove = function(event) {
                if (
                    clickMouseX === undefined ||
                    clickMouseY === undefined ||
                    clickTranslateX === undefined ||
                    clickTranslateY === undefined
                )
                    return;

                // offset image based on original and current mouse position
                translateX = clickTranslateX + window.mouseX - clickMouseX;
                translateY = clickTranslateY + window.mouseY - clickMouseY;
                updateTransform();
                event.preventDefault();

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('mousemove', onWindowMouseMove);
            };

            // when window is resized
            const onWindowResize = function(event) {
                fit();

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('resize', onWindowResize);
            };

            // attach the necessary event listeners
            img.addEventListener('dblclick', fit);
            img.addEventListener('mousedown', onImageMouseDown);
            container.addEventListener('wheel', onContainerWheel);
            container.addEventListener('mousedown', onContainerClick);
            container.addEventListener('touchstart', onContainerClick);
            window.addEventListener('mouseup', onWindowMouseUp);
            window.addEventListener('mousemove', onWindowMouseMove);
            window.addEventListener('resize', onWindowResize);

            // run fit() after lightbox atttached to document and <img> Loaded
            // so needed container and img dimensions available
            img.addEventListener('load', fit);

            return img;
        }

        // make caption
        function makeCaption(img) {
            const caption = document.createElement('div');
            caption.id = 'lightbox_caption';
            const captionSource = img.nextElementSibling;
            if (captionSource.tagName.toLowerCase() === 'figcaption') {
                const captionCopy = makeCopy(captionSource);
                caption.innerHTML = captionCopy.innerHTML;
            }

            caption.addEventListener('touchstart', function(event) {
                event.stopPropagation();
            });

            return caption;
        }

        // make carbon copy of html dom element
        function makeCopy(source) {
            const sourceCopy = source.cloneNode(true);

            // delete elements marked with ignore (eg anchor and jump buttons)
            const deleteFromCopy = sourceCopy.querySelectorAll(
                '[data-ignore="true"]'
            );
            for (const element of deleteFromCopy)
                element.remove();

            // delete certain element attributes
            const attributes = [
                'id',
                'data-collapsed',
                'data-selected',
                'data-highlighted',
                'data-glow'
            ];
            for (const attribute of attributes) {
                sourceCopy.removeAttribute(attribute);
                const elements = sourceCopy.querySelectorAll(
                    '[' + attribute + ']'
                );
                for (const element of elements)
                    element.removeAttribute(attribute);
            }

            return sourceCopy;
        }

        // make button to jump to previous image in document
        function makePrevButton(img) {
            const prevButton = document.createElement('button');
            prevButton.id = 'lightbox_prev_button';
            prevButton.title = 'Jump to the previous image in the document [←]';
            prevButton.classList.add('icon_button', 'lightbox_button');
            prevButton.innerHTML = document.querySelector(
                '.icon_caret_left'
            ).innerHTML;

            // attach click listeners to button
            prevButton.addEventListener('click', function() {
                getPrevImg(img).click();
            });

            return prevButton;
        }

        // make button to jump to next image in document
        function makeNextButton(img) {
            const nextButton = document.createElement('button');
            nextButton.id = 'lightbox_next_button';
            nextButton.title = 'Jump to the next image in the document [→]';
            nextButton.classList.add('icon_button', 'lightbox_button');
            nextButton.innerHTML = document.querySelector(
                '.icon_caret_right'
            ).innerHTML;

            // attach click listeners to button
            nextButton.addEventListener('click', function() {
                getNextImg(img).click();
            });

            return nextButton;
        }

        // get previous image in document
        function getPrevImg(img) {
            const imgs = document.querySelectorAll('.lightbox_document_img');

            // find index of provided img
            let index;
            for (index = 0; index < imgs.length; index++) {
                if (imgs[index] === img)
                    break;
            }


            // wrap index to other side if < 1
            if (index - 1 >= 0)
                index--;
            else
                index = imgs.length - 1;
            return imgs[index];
        }

        // get next image in document
        function getNextImg(img) {
            const imgs = document.querySelectorAll('.lightbox_document_img');

            // find index of provided img
            let index;
            for (index = 0; index < imgs.length; index++) {
                if (imgs[index] === img)
                    break;
            }


            // wrap index to other side if > total
            if (index + 1 <= imgs.length - 1)
                index++;
            else
                index = 0;
            return imgs[index];
        }

        // close lightbox
        function closeLightbox() {
            focusBody();

            const lightbox = document.getElementById('lightbox_overlay');
            if (lightbox)
                lightbox.remove();
        }

        // make all elements behind lightbox non-focusable
        function blurBody(overlay) {
            const all = document.querySelectorAll('*');
            for (const element of all)
                element.tabIndex = -1;
            document.body.classList.add('body_no_scroll');
        }

        // make all elements focusable again
        function focusBody() {
            const all = document.querySelectorAll('*');
            for (const element of all)
                element.removeAttribute('tabIndex');
            document.body.classList.remove('body_no_scroll');
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
    <!-- modified from: https://fontawesome.com/icons/caret-left -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
        ></path>
    </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
    <!-- modified from: https://fontawesome.com/icons/caret-right -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
        ></path>
    </svg>
</template>
<!-- math plugin configuration -->

<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
        "CommonHTML": { linebreaks: { automatic: true } },
        "HTML-CSS": { linebreaks: { automatic: true } },
        "SVG": { linebreaks: { automatic: true } }
    });
</script>

<!-- math plugin -->

<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML'>
    // /////////////////////////
    // DESCRIPTION
    // /////////////////////////

    // This third-party plugin 'MathJax' allows the proper rendering of
    // math/equations written in LaTeX.

    // https://www.mathjax.org/
</script>
<!-- annotations plugin configuration -->

<script>
    window.hypothesisConfig = function() {
        return {
            branding: {
                accentColor: '#2196f3',
                appBackgroundColor: '#f8f8f8',
                ctaBackgroundColor: '#f8f8f8',
                ctaTextColor: '#000000',
                selectionFontFamily: 'Open Sans, Helvetica, sans serif',
                annotationFontFamily: 'Open Sans, Helvetica, sans serif'
            }
        };
    };
</script>

<!-- annotations plugin -->

<script src='https://hypothes.is/embed.js'>
    // /////////////////////////
    // DESCRIPTION
    // /////////////////////////

    // This third-party plugin 'Hypothesis' allows public annotation of the
    // manuscript.

    // https://web.hypothes.is/
</script>
<!-- analytics plugin -->

<!-- copy and paste code from Google Analytics or similar service here -->
</body>
</html>
